// @clerk/localizations/dist/ko-KR.mjs
// src/ko-KR.ts
var koKR = {
  locale: "ko-KR",
  apiKeys: {
    action__add: "\uC0C8 \uD0A4 \uB9CC\uB4E4\uAE30",
    action__search: "\uD0A4 \uAC80\uC0C9",
    copySecret: {
      formButtonPrimary__copyAndClose: "\uBCF5\uC0AC\uD558\uACE0 \uB2EB\uAE30",
      formHint: "\uBCF4\uC548\uC0C1 \uB098\uC911\uC5D0 \uB2E4\uC2DC \uBCFC \uC218 \uC5C6\uC5B4\uC694.",
      formTitle: '\uC9C0\uAE08 "{{name}}" API \uD0A4\uB97C \uBCF5\uC0AC\uD574 \uC8FC\uC138\uC694'
    },
    createdAndExpirationStatus__expiresOn: "\uC0DD\uC131 {{ createdDate | shortDate('ko-KR') }} \u2022 \uB9CC\uB8CC {{ expiresDate | longDate('ko-KR') }}",
    createdAndExpirationStatus__never: "\uC0DD\uC131 {{ createdDate | shortDate('ko-KR') }} \u2022 \uB9CC\uB8CC \uC5C6\uC74C",
    detailsTitle__emptyRow: "API \uD0A4\uAC00 \uC5C6\uC5B4\uC694",
    formButtonPrimary__add: "\uD0A4 \uB9CC\uB4E4\uAE30",
    formFieldCaption__expiration__expiresOn: "{{ date }}\uC5D0 \uB9CC\uB8CC\uB3FC\uC694",
    formFieldCaption__expiration__never: "\uC774 \uD0A4\uB294 \uB9CC\uB8CC\uB418\uC9C0 \uC54A\uC544\uC694",
    formFieldOption__expiration__180d: "180\uC77C",
    formFieldOption__expiration__1d: "1\uC77C",
    formFieldOption__expiration__1y: "1\uB144",
    formFieldOption__expiration__30d: "30\uC77C",
    formFieldOption__expiration__60d: "60\uC77C",
    formFieldOption__expiration__7d: "7\uC77C",
    formFieldOption__expiration__90d: "90\uC77C",
    formFieldOption__expiration__never: "\uB9CC\uB8CC \uC5C6\uC74C",
    formHint: "\uC774\uB984\uC744 \uC785\uB825\uD574 \uC0C8 \uD0A4\uB97C \uB9CC\uB4E4\uC5B4\uC694. \uD544\uC694\uD558\uBA74 \uC5B8\uC81C\uB4E0\uC9C0 \uD3D0\uAE30\uD560 \uC218 \uC788\uC5B4\uC694.",
    formTitle: "\uC0C8 API \uD0A4 \uCD94\uAC00",
    lastUsed__days: "{{days}}\uC77C \uC804",
    lastUsed__hours: "{{hours}}\uC2DC\uAC04 \uC804",
    lastUsed__minutes: "{{minutes}}\uBD84 \uC804",
    lastUsed__months: "{{months}}\uAC1C\uC6D4 \uC804",
    lastUsed__seconds: "{{seconds}}\uCD08 \uC804",
    lastUsed__years: "{{years}}\uB144 \uC804",
    menuAction__revoke: "\uD0A4 \uD3D0\uAE30",
    revokeConfirmation: {
      confirmationText: "\uD3D0\uAE30",
      formButtonPrimary__revoke: "\uD0A4 \uD3D0\uAE30",
      formHint: "\uC774 \uC2DC\uD06C\uB9BF \uD0A4\uB97C \uC0AD\uC81C\uD560\uAE4C\uC694?",
      formTitle: '"{{apiKeyName}}" \uC2DC\uD06C\uB9BF \uD0A4\uB97C \uD3D0\uAE30\uD560\uAE4C\uC694?'
    }
  },
  backButton: "\uB3CC\uC544\uAC00\uAE30",
  badge__activePlan: "\uD65C\uC131",
  badge__canceledEndsAt: "\uCDE8\uC18C\uB428 \u2022 {{ date | shortDate('ko-KR') }}\uC5D0 \uC885\uB8CC",
  badge__currentPlan: "\uD604\uC7AC \uD50C\uB79C",
  badge__default: "\uAE30\uBCF8\uAC12",
  badge__endsAt: "{{ date | shortDate('ko-KR') }}\uC5D0 \uC885\uB8CC",
  badge__expired: "\uB9CC\uB8CC\uB428",
  badge__freeTrial: "\uBB34\uB8CC \uCCB4\uD5D8",
  badge__otherImpersonatorDevice: "\uAE30\uD0C0 \uC0AC\uCE6D \uC7A5\uCE58",
  badge__pastDueAt: "{{ date | shortDate('ko-KR') }}\uC5D0 \uC5F0\uCCB4",
  badge__pastDuePlan: "\uC5F0\uCCB4\uB428",
  badge__primary: "\uAE30\uBCF8",
  badge__renewsAt: "{{ date | shortDate('ko-KR') }}\uC5D0 \uAC31\uC2E0",
  badge__requiresAction: "\uC870\uCE58 \uD544\uC694",
  badge__startsAt: "{{ date | shortDate('ko-KR') }}\uC5D0 \uC2DC\uC791",
  badge__thisDevice: "\uC774 \uC7A5\uCE58",
  badge__trialEndsAt: "{{ date | shortDate('ko-KR') }}\uC5D0 \uCCB4\uD5D8 \uC885\uB8CC",
  badge__unverified: "\uBBF8\uD655\uC778",
  badge__upcomingPlan: "\uC608\uC815\uB41C \uD50C\uB79C",
  badge__userDevice: "\uC0AC\uC6A9\uC790 \uC7A5\uCE58",
  badge__you: "\uB098",
  billing: {
    addPaymentMethod__label: "\uACB0\uC81C \uC218\uB2E8 \uCD94\uAC00",
    alwaysFree: "\uD56D\uC0C1 \uBB34\uB8CC",
    annually: "\uC5F0\uAC04",
    availableFeatures: "\uC0AC\uC6A9 \uAC00\uB2A5\uD55C \uAE30\uB2A5",
    billedAnnually: "\uC5F0\uAC04 \uACB0\uC81C",
    billedMonthlyOnly: "\uC6D4\uAC04 \uACB0\uC81C\uB9CC \uAC00\uB2A5",
    cancelFreeTrial: "\uBB34\uB8CC \uCCB4\uD5D8 \uCDE8\uC18C",
    cancelFreeTrialAccessUntil: "\uBB34\uB8CC \uCCB4\uD5D8\uC740 {{ date | longDate('ko-KR') }}\uAE4C\uC9C0 \uC720\uC9C0\uB3FC\uC694. \uC774\uD6C4\uC5D0\uB294 \uCCB4\uD5D8 \uAE30\uB2A5\uC744 \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uC5B4\uC694. \uC694\uAE08\uC740 \uCCAD\uAD6C\uB418\uC9C0 \uC54A\uC544\uC694.",
    cancelFreeTrialTitle: "{{plan}} \uD50C\uB79C \uBB34\uB8CC \uCCB4\uD5D8\uC744 \uCDE8\uC18C\uD560\uAE4C\uC694?",
    cancelSubscription: "\uAD6C\uB3C5 \uCDE8\uC18C",
    cancelSubscriptionAccessUntil: "{{ date | longDate('ko-KR') }}\uAE4C\uC9C0 '{{plan}}' \uAE30\uB2A5\uC744 \uC0AC\uC6A9\uD560 \uC218 \uC788\uC5B4\uC694. \uC774\uD6C4\uC5D0\uB294 \uC774\uC6A9\uD560 \uC218 \uC5C6\uC5B4\uC694.",
    cancelSubscriptionNoCharge: "\uC774 \uAD6C\uB3C5\uC5D0\uB294 \uC694\uAE08\uC774 \uCCAD\uAD6C\uB418\uC9C0 \uC54A\uC544\uC694.",
    cancelSubscriptionPastDue: "\uAD6C\uB3C5\uC774 \uC989\uC2DC \uC885\uB8CC\uB418\uACE0 \uBAA8\uB4E0 \uD50C\uB79C \uAE30\uB2A5\uC744 \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uC5B4\uC694. \uB2E4\uC74C \uACB0\uC81C \uC2DC \uC5F0\uCCB4 \uAE08\uC561\uC744 \uACB0\uC81C\uD558\uC154\uC57C \uD574\uC694.",
    cancelSubscriptionTitle: "{{plan}} \uAD6C\uB3C5\uC744 \uCDE8\uC18C\uD560\uAE4C\uC694?",
    cannotSubscribeMonthly: "\uC774 \uD50C\uB79C\uC740 \uC6D4\uAC04 \uACB0\uC81C\uAC00 \uBD88\uAC00\uD574\uC694. \uC5F0\uAC04 \uACB0\uC81C\uB97C \uC120\uD0DD\uD574 \uC8FC\uC138\uC694.",
    cannotSubscribeUnrecoverable: "\uC774 \uD50C\uB79C\uC73C\uB85C \uAD6C\uB3C5\uD560 \uC218 \uC5C6\uC5B4\uC694. \uD604\uC7AC \uAD6C\uB3C5\uC774 \uB354 \uB192\uC740 \uC694\uAE08\uC81C\uC608\uC694.",
    checkout: {
      description__paymentSuccessful: "\uACB0\uC81C\uAC00 \uC644\uB8CC\uB410\uC5B4\uC694.",
      description__subscriptionSuccessful: "\uC0C8 \uAD6C\uB3C5\uC774 \uC900\uBE44\uB410\uC5B4\uC694.",
      downgradeNotice: "\uD604\uC7AC \uAD6C\uB3C5\uC740 \uACB0\uC81C \uC8FC\uAE30 \uC885\uB8CC\uAE4C\uC9C0 \uC720\uC9C0\uB418\uACE0, \uC774\uD6C4 \uC774 \uAD6C\uB3C5\uC73C\uB85C \uC804\uD658\uB3FC\uC694.",
      emailForm: {
        subtitle: "\uACB0\uC81C\uB97C \uC644\uB8CC\uD558\uB824\uBA74 \uC601\uC218\uC99D\uC744 \uBC1B\uC744 \uC774\uBA54\uC77C \uC8FC\uC18C\uB97C \uCD94\uAC00\uD574\uC57C \uD574\uC694.",
        title: "\uC774\uBA54\uC77C \uC8FC\uC18C \uCD94\uAC00"
      },
      lineItems: {
        title__freeTrialEndsAt: "\uBB34\uB8CC \uCCB4\uD5D8 \uC885\uB8CC\uC77C",
        title__paymentMethod: "\uACB0\uC81C \uC218\uB2E8",
        title__statementId: "\uBA85\uC138\uC11C ID",
        title__subscriptionBegins: "\uAD6C\uB3C5 \uC2DC\uC791",
        title__totalPaid: "\uCD1D \uACB0\uC81C"
      },
      pastDueNotice: "\uC774\uC804 \uAD6C\uB3C5\uC774 \uC5F0\uCCB4\uB418\uC5B4 \uACB0\uC81C\uAC00 \uB418\uC9C0 \uC54A\uC558\uC5B4\uC694.",
      perMonth: "\uC6D4",
      title: "\uACB0\uC81C",
      title__paymentSuccessful: "\uACB0\uC81C\uAC00 \uC644\uB8CC\uB410\uC5B4\uC694!",
      title__subscriptionSuccessful: "\uC131\uACF5!",
      title__trialSuccess: "\uBB34\uB8CC \uCCB4\uD5D8\uC774 \uC2DC\uC791\uB410\uC5B4\uC694!",
      totalDueAfterTrial: "{{days}}\uC77C \uD6C4 \uBB34\uB8CC \uCCB4\uD5D8\uC774 \uB05D\uB098\uBA74 \uACB0\uC81C\uD560 \uAE08\uC561"
    },
    credit: "\uD06C\uB808\uB527",
    creditRemainder: "\uD604\uC7AC \uAD6C\uB3C5 \uB0A8\uC740 \uAE30\uAC04\uC5D0 \uB300\uD55C \uD06C\uB808\uB527",
    defaultFreePlanActive: "\uD604\uC7AC \uBB34\uB8CC \uD50C\uB79C\uC744 \uC0AC\uC6A9 \uC911\uC774\uC5D0\uC694",
    free: "\uBB34\uB8CC",
    getStarted: "\uC2DC\uC791\uD558\uAE30",
    keepFreeTrial: "\uBB34\uB8CC \uCCB4\uD5D8 \uC720\uC9C0",
    keepSubscription: "\uAD6C\uB3C5 \uC720\uC9C0",
    manage: "\uAD00\uB9AC",
    manageSubscription: "\uAD6C\uB3C5 \uAD00\uB9AC",
    month: "\uC6D4",
    monthly: "\uC6D4\uAC04",
    pastDue: "\uC5F0\uCCB4",
    pay: "{{amount}} \uACB0\uC81C",
    paymentMethod: {
      applePayDescription: {
        annual: "\uC5F0\uAC04 \uACB0\uC81C",
        monthly: "\uC6D4\uAC04 \uACB0\uC81C"
      },
      dev: {
        anyNumbers: "\uC544\uBB34 \uC22B\uC790\uB098",
        cardNumber: "\uCE74\uB4DC \uBC88\uD638",
        cvcZip: "CVC, \uC6B0\uD3B8\uBC88\uD638",
        developmentMode: "\uAC1C\uBC1C \uBAA8\uB4DC",
        expirationDate: "\uB9CC\uB8CC\uC77C",
        testCardInfo: "\uD14C\uC2A4\uD2B8 \uCE74\uB4DC \uC815\uBCF4"
      }
    },
    paymentMethods__label: "\uACB0\uC81C \uC218\uB2E8",
    popular: "\uC778\uAE30",
    pricingTable: {
      billingCycle: "\uACB0\uC81C \uC8FC\uAE30",
      included: "\uD3EC\uD568"
    },
    reSubscribe: "\uB2E4\uC2DC \uAD6C\uB3C5",
    seeAllFeatures: "\uC804\uCCB4 \uAE30\uB2A5 \uBCF4\uAE30",
    startFreeTrial: "\uBB34\uB8CC \uCCB4\uD5D8 \uC2DC\uC791",
    startFreeTrial__days: "{{days}}\uC77C \uBB34\uB8CC \uCCB4\uD5D8 \uC2DC\uC791",
    subscribe: "\uAD6C\uB3C5\uD558\uAE30",
    subscriptionDetails: {
      beginsOn: "\uC2DC\uC791\uC77C",
      currentBillingCycle: "\uD604\uC7AC \uACB0\uC81C \uC8FC\uAE30",
      endsOn: "\uC885\uB8CC\uC77C",
      firstPaymentAmount: "\uCCAB \uACB0\uC81C \uAE08\uC561",
      firstPaymentOn: "\uCCAB \uACB0\uC81C\uC77C",
      nextPaymentAmount: "\uB2E4\uC74C \uACB0\uC81C \uAE08\uC561",
      nextPaymentOn: "\uB2E4\uC74C \uACB0\uC81C\uC77C",
      pastDueAt: "\uC5F0\uCCB4\uC77C",
      renewsAt: "\uAC31\uC2E0\uC77C",
      subscribedOn: "\uAD6C\uB3C5 \uC2DC\uC791\uC77C",
      title: "\uAD6C\uB3C5",
      trialEndsOn: "\uCCB4\uD5D8 \uC885\uB8CC\uC77C",
      trialStartedOn: "\uCCB4\uD5D8 \uC2DC\uC791\uC77C"
    },
    subtotal: "\uC18C\uACC4",
    switchPlan: "\uC774 \uD50C\uB79C\uC73C\uB85C \uC804\uD658",
    switchToAnnual: "\uC5F0\uAC04 \uACB0\uC81C\uB85C \uBCC0\uACBD",
    switchToAnnualWithAnnualPrice: "\uC5F0\uAC04 {{currency}}{{price}} / \uB144\uC73C\uB85C \uBCC0\uACBD",
    switchToMonthly: "\uC6D4\uAC04 \uACB0\uC81C\uB85C \uBCC0\uACBD",
    switchToMonthlyWithPrice: "\uC6D4\uAC04 {{currency}}{{price}} / \uC6D4\uB85C \uBCC0\uACBD",
    totalDue: "\uCD1D \uACB0\uC81C \uAE08\uC561",
    totalDueToday: "\uC624\uB298 \uACB0\uC81C \uAE08\uC561",
    viewFeatures: "\uAE30\uB2A5 \uBCF4\uAE30",
    viewPayment: "\uACB0\uC81C \uBCF4\uAE30",
    year: "\uB144"
  },
  createOrganization: {
    formButtonSubmit: "\uC870\uC9C1 \uB9CC\uB4E4\uAE30",
    invitePage: {
      formButtonReset: "\uB118\uAE30\uAE30"
    },
    title: "\uC870\uC9C1 \uB9CC\uB4E4\uAE30"
  },
  dates: {
    lastDay: "\uC5B4\uC81C {{ date | timeString('ko-KR') }}",
    next6Days: "\uB2E4\uC74C {{ date | weekday('ko-KR','long') }} {{ date | timeString('ko-KR') }}",
    nextDay: "\uB0B4\uC77C {{ date | timeString('ko-KR') }}",
    numeric: "{{ date | numeric('ko-KR') }}",
    previous6Days: "\uC9C0\uB09C {{ date | weekday('ko-KR','long') }} {{ date | timeString('ko-KR') }}",
    sameDay: "\uC624\uB298 {{ date | timeString('ko-KR') }}"
  },
  dividerText: "\uB610\uB294",
  footerActionLink__alternativePhoneCodeProvider: "\uB300\uC2E0 SMS\uB85C \uCF54\uB4DC \uBC1B\uAE30",
  footerActionLink__useAnotherMethod: "\uB2E4\uB978 \uBC29\uBC95 \uC0AC\uC6A9\uD558\uAE30",
  footerPageLink__help: "\uB3C4\uC6C0",
  footerPageLink__privacy: "\uAC1C\uC778\uC815\uBCF4\uCC98\uB9AC\uBC29\uCE68",
  footerPageLink__terms: "\uC57D\uAD00",
  formButtonPrimary: "\uACC4\uC18D",
  formButtonPrimary__verify: "\uD655\uC778\uD558\uAE30",
  formFieldAction__forgotPassword: "\uBE44\uBC00\uBC88\uD638\uB97C \uC78A\uC73C\uC168\uB098\uC694?",
  formFieldError__matchingPasswords: "\uBE44\uBC00\uBC88\uD638\uAC00 \uC77C\uCE58\uD574\uC694.",
  formFieldError__notMatchingPasswords: "\uBE44\uBC00\uBC88\uD638\uAC00 \uC77C\uCE58\uD558\uC9C0 \uC54A\uC544\uC694.",
  formFieldError__verificationLinkExpired: "\uC778\uC99D \uB9C1\uD06C\uAC00 \uB9CC\uB8CC\uB410\uC5B4\uC694. \uC0C8 \uB9C1\uD06C\uB97C \uC694\uCCAD\uD574 \uC8FC\uC138\uC694.",
  formFieldHintText__optional: "\uC120\uD0DD\uC0AC\uD56D",
  formFieldHintText__slug: "\uC2AC\uB7EC\uADF8\uB294 \uC0AC\uB78C\uC774 \uC77D\uAE30 \uC26C\uC6B4 \uACE0\uC720 ID\uC608\uC694. \uBCF4\uD1B5 URL\uC5D0 \uC0AC\uC6A9\uB3FC\uC694.",
  formFieldInputPlaceholder__apiKeyDescription: "\uC774 \uD0A4\uB97C \uB9CC\uB4DC\uB294 \uC774\uC720\uB97C \uC801\uC5B4\uC8FC\uC138\uC694",
  formFieldInputPlaceholder__apiKeyExpirationDate: "\uB0A0\uC9DC\uB97C \uC120\uD0DD\uD558\uC138\uC694",
  formFieldInputPlaceholder__apiKeyName: "\uC2DC\uD06C\uB9BF \uD0A4 \uC774\uB984\uC744 \uC785\uB825\uD558\uC138\uC694",
  formFieldInputPlaceholder__backupCode: "\uBC31\uC5C5 \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "\uACC4\uC815 \uC0AD\uC81C",
  formFieldInputPlaceholder__emailAddress: "\uC774\uBA54\uC77C \uC8FC\uC18C\uB97C \uC785\uB825\uD558\uC138\uC694",
  formFieldInputPlaceholder__emailAddress_username: "\uC774\uBA54\uC77C \uC8FC\uC18C \uB610\uB294 \uC544\uC774\uB514\uB97C \uC785\uB825\uD558\uC138\uC694",
  formFieldInputPlaceholder__emailAddresses: "\uC774\uBA54\uC77C \uC8FC\uC18C\uB97C \uACF5\uBC31\uC774\uB098 \uC27C\uD45C\uB85C \uAD6C\uBD84\uD574 \uC785\uB825\uD558\uAC70\uB098 \uBD99\uC5EC\uB123\uC5B4 \uC8FC\uC138\uC694",
  formFieldInputPlaceholder__firstName: "\uC774\uB984",
  formFieldInputPlaceholder__lastName: "\uC131",
  formFieldInputPlaceholder__organizationDomain: "example.com",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "you@example.com",
  formFieldInputPlaceholder__organizationName: "\uC870\uC9C1 \uC774\uB984",
  formFieldInputPlaceholder__organizationSlug: "my-org",
  formFieldInputPlaceholder__password: "\uBE44\uBC00\uBC88\uD638\uB97C \uC785\uB825\uD558\uC138\uC694",
  formFieldInputPlaceholder__phoneNumber: "\uD734\uB300\uD3F0 \uBC88\uD638\uB97C \uC785\uB825\uD558\uC138\uC694",
  formFieldInputPlaceholder__username: void 0,
  formFieldInput__emailAddress_format: "\uC608\uC2DC: name@example.com",
  formFieldLabel__apiKey: "API \uD0A4",
  formFieldLabel__apiKeyDescription: "\uC124\uBA85",
  formFieldLabel__apiKeyExpiration: "\uB9CC\uB8CC",
  formFieldLabel__apiKeyName: "\uC2DC\uD06C\uB9BF \uD0A4 \uC774\uB984",
  formFieldLabel__automaticInvitations: "\uC774 \uB3C4\uBA54\uC778\uC5D0 \uC790\uB3D9 \uCD08\uB300 \uC0AC\uC6A9",
  formFieldLabel__backupCode: "\uBC31\uC5C5 \uCF54\uB4DC",
  formFieldLabel__confirmDeletion: "\uD655\uC778",
  formFieldLabel__confirmPassword: "\uBE44\uBC00\uBC88\uD638 \uD655\uC778",
  formFieldLabel__currentPassword: "\uD604\uC7AC \uBE44\uBC00\uBC88\uD638",
  formFieldLabel__emailAddress: "\uC774\uBA54\uC77C \uC8FC\uC18C",
  formFieldLabel__emailAddress_username: "\uC774\uBA54\uC77C \uC8FC\uC18C \uD639\uC740 \uC544\uC774\uB514",
  formFieldLabel__emailAddresses: "\uC774\uBA54\uC77C \uC8FC\uC18C",
  formFieldLabel__firstName: "\uC774\uB984",
  formFieldLabel__lastName: "\uC131",
  formFieldLabel__newPassword: "\uC0C8 \uBE44\uBC00\uBC88\uD638",
  formFieldLabel__organizationDomain: "\uB3C4\uBA54\uC778",
  formFieldLabel__organizationDomainDeletePending: "\uB300\uAE30 \uC911\uC778 \uCD08\uB300\uC640 \uC81C\uC548 \uC0AD\uC81C",
  formFieldLabel__organizationDomainEmailAddress: "\uC778\uC99D \uC774\uBA54\uC77C \uC8FC\uC18C",
  formFieldLabel__organizationDomainEmailAddressDescription: "\uC774 \uB3C4\uBA54\uC778\uC758 \uC774\uBA54\uC77C \uC8FC\uC18C\uB97C \uC785\uB825\uD558\uBA74 \uCF54\uB4DC\uB97C \uBC1B\uC544 \uB3C4\uBA54\uC778\uC744 \uC778\uC99D\uD560 \uC218 \uC788\uC5B4\uC694.",
  formFieldLabel__organizationName: "\uC774\uB984",
  formFieldLabel__organizationSlug: "\uC2AC\uB7EC\uADF8",
  formFieldLabel__passkeyName: "\uD328\uC2A4\uD0A4 \uC774\uB984",
  formFieldLabel__password: "\uBE44\uBC00\uBC88\uD638",
  formFieldLabel__phoneNumber: "\uD734\uB300\uD3F0 \uBC88\uD638",
  formFieldLabel__role: "\uC5ED\uD560",
  formFieldLabel__signOutOfOtherSessions: "\uB2E4\uB978 \uBAA8\uB4E0 \uAE30\uAE30\uC5D0\uC11C \uB85C\uADF8\uC544\uC6C3",
  formFieldLabel__username: "\uC544\uC774\uB514",
  impersonationFab: {
    action__signOut: "\uB85C\uADF8\uC544\uC6C3",
    title: "{{identifier}}\uB85C \uB85C\uADF8\uC778\uD588\uC5B4\uC694"
  },
  lastAuthenticationStrategy: "\uCD5C\uADFC \uC0AC\uC6A9",
  maintenanceMode: "\uC9C0\uAE08 \uC810\uAC80 \uC911\uC774\uC5D0\uC694. \uAC71\uC815 \uB9C8\uC138\uC694, \uBA87 \uBD84\uC774\uBA74 \uB05D\uB0A0 \uAC70\uC608\uC694.",
  membershipRole__admin: "\uAD00\uB9AC\uC790",
  membershipRole__basicMember: "\uBA64\uBC84",
  membershipRole__guestMember: "\uAC8C\uC2A4\uD2B8",
  organizationList: {
    action__createOrganization: "\uC870\uC9C1 \uB9CC\uB4E4\uAE30",
    action__invitationAccept: "\uCC38\uC5EC",
    action__suggestionsAccept: "\uCC38\uC5EC \uC694\uCCAD",
    createOrganization: "\uC870\uC9C1 \uB9CC\uB4E4\uAE30",
    invitationAcceptedLabel: "\uCC38\uC5EC\uD568",
    subtitle: "{{applicationName}}\uB85C \uACC4\uC18D\uD558\uB824\uBA74",
    suggestionsAcceptedLabel: "\uC2B9\uC778 \uB300\uAE30",
    title: "\uACC4\uC815 \uC120\uD0DD",
    titleWithoutPersonal: "\uC870\uC9C1 \uC120\uD0DD"
  },
  organizationProfile: {
    apiKeysPage: {
      title: "API \uD0A4"
    },
    badge__automaticInvitation: "\uC790\uB3D9 \uCD08\uB300",
    badge__automaticSuggestion: "\uC790\uB3D9 \uC81C\uC548",
    badge__manualInvitation: "\uC790\uB3D9 \uB4F1\uB85D \uC5C6\uC74C",
    badge__unverified: "\uBBF8\uC778\uC99D",
    billingPage: {
      paymentHistorySection: {
        empty: "\uACB0\uC81C \uB0B4\uC5ED\uC774 \uC5C6\uC5B4\uC694",
        notFound: "\uACB0\uC81C \uC2DC\uB3C4\uB97C \uCC3E\uC744 \uC218 \uC5C6\uC5B4\uC694",
        tableHeader__amount: "\uAE08\uC561",
        tableHeader__date: "\uB0A0\uC9DC",
        tableHeader__status: "\uC0C1\uD0DC"
      },
      paymentMethodsSection: {
        actionLabel__default: "\uAE30\uBCF8\uC73C\uB85C \uC124\uC815",
        actionLabel__remove: "\uC0AD\uC81C",
        add: "\uC0C8 \uACB0\uC81C \uC218\uB2E8 \uCD94\uAC00",
        addSubtitle: "\uACC4\uC815\uC5D0 \uC0C8 \uACB0\uC81C \uC218\uB2E8\uC744 \uCD94\uAC00\uD558\uC138\uC694.",
        cancelButton: "\uCDE8\uC18C",
        formButtonPrimary__add: "\uACB0\uC81C \uC218\uB2E8 \uCD94\uAC00",
        formButtonPrimary__pay: "{{amount}} \uACB0\uC81C",
        payWithTestCardButton: "\uD14C\uC2A4\uD2B8 \uCE74\uB4DC\uB85C \uACB0\uC81C",
        removeMethod: {
          messageLine1: "{{identifier}}\uAC00 \uC774 \uACC4\uC815\uC5D0\uC11C \uC0AD\uC81C\uB3FC\uC694.",
          messageLine2: "\uC774 \uACB0\uC81C \uC218\uB2E8\uC740 \uB354 \uC774\uC0C1 \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uACE0, \uC5F0\uACB0\uB41C \uAD6C\uB3C5\uB3C4 \uC791\uB3D9\uD558\uC9C0 \uC54A\uC544\uC694.",
          successMessage: "{{paymentMethod}}\uC774(\uAC00) \uACC4\uC815\uC5D0\uC11C \uC0AD\uC81C\uB410\uC5B4\uC694.",
          title: "\uACB0\uC81C \uC218\uB2E8 \uC0AD\uC81C"
        },
        title: "\uACB0\uC81C \uC218\uB2E8"
      },
      start: {
        headerTitle__payments: "\uACB0\uC81C",
        headerTitle__plans: "\uD50C\uB79C",
        headerTitle__statements: "\uBA85\uC138\uC11C",
        headerTitle__subscriptions: "\uAD6C\uB3C5"
      },
      statementsSection: {
        empty: "\uD45C\uC2DC\uD560 \uBA85\uC138\uC11C\uAC00 \uC5C6\uC5B4\uC694",
        itemCaption__paidForPlan: "{{plan}} {{period}} \uD50C\uB79C \uACB0\uC81C",
        itemCaption__proratedCredit: "\uC774\uC804 \uAD6C\uB3C5 \uC77C\uBD80 \uC0AC\uC6A9\uC5D0 \uB300\uD55C \uBE44\uB840 \uD06C\uB808\uB527",
        itemCaption__subscribedAndPaidForPlan: "{{plan}} {{period}} \uD50C\uB79C\uC744 \uAD6C\uB3C5\uD558\uACE0 \uACB0\uC81C\uD588\uC5B4\uC694",
        notFound: "\uBA85\uC138\uC11C\uB97C \uCC3E\uC744 \uC218 \uC5C6\uC5B4\uC694",
        tableHeader__amount: "\uAE08\uC561",
        tableHeader__date: "\uB0A0\uC9DC",
        title: "\uBA85\uC138\uC11C",
        totalPaid: "\uCD1D \uACB0\uC81C"
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: "\uAD00\uB9AC",
        actionLabel__newSubscription: "\uD50C\uB79C \uAD6C\uB3C5\uD558\uAE30",
        actionLabel__switchPlan: "\uD50C\uB79C \uBCC0\uACBD",
        tableHeader__edit: "\uD3B8\uC9D1",
        tableHeader__plan: "\uD50C\uB79C",
        tableHeader__startDate: "\uC2DC\uC791\uC77C",
        title: "\uAD6C\uB3C5"
      },
      subscriptionsSection: {
        actionLabel__default: "\uAD00\uB9AC"
      },
      switchPlansSection: {
        title: "\uD50C\uB79C \uBCC0\uACBD"
      },
      title: "\uACB0\uC81C"
    },
    createDomainPage: {
      subtitle: "\uB3C4\uBA54\uC778\uC744 \uCD94\uAC00\uD574 \uC778\uC99D\uD558\uC138\uC694. \uC774 \uB3C4\uBA54\uC778\uC758 \uC774\uBA54\uC77C\uC744 \uAC00\uC9C4 \uC0AC\uC6A9\uC790\uB294 \uC870\uC9C1\uC5D0 \uC790\uB3D9\uC73C\uB85C \uCC38\uC5EC\uD558\uAC70\uB098 \uCC38\uC5EC \uC694\uCCAD\uC744 \uD560 \uC218 \uC788\uC5B4\uC694.",
      title: "\uB3C4\uBA54\uC778 \uCD94\uAC00"
    },
    invitePage: {
      detailsTitle__inviteFailed: "\uCD08\uB300\uB97C \uBCF4\uB0BC \uC218 \uC5C6\uC5B4\uC694. \uB2E4\uC74C \uC774\uBA54\uC77C \uC8FC\uC18C\uC5D0\uB294 \uC774\uBBF8 \uB300\uAE30 \uC911\uC778 \uCD08\uB300\uAC00 \uC788\uC5B4\uC694: {{email_addresses}}.",
      formButtonPrimary__continue: "\uCD08\uB300 \uBCF4\uB0B4\uAE30",
      selectDropdown__role: "\uC5ED\uD560 \uC120\uD0DD",
      subtitle: "\uACF5\uBC31\uC774\uB098 \uC27C\uD45C\uB85C \uAD6C\uBD84\uD574 \uC774\uBA54\uC77C \uC8FC\uC18C\uB97C \uC785\uB825\uD558\uAC70\uB098 \uBD99\uC5EC\uB123\uC5B4 \uC8FC\uC138\uC694.",
      successMessage: "\uCD08\uB300\uAC00 \uC131\uACF5\uC801\uC73C\uB85C \uC804\uC1A1\uB410\uC5B4\uC694",
      title: "\uC0C8 \uBA64\uBC84 \uCD08\uB300"
    },
    membersPage: {
      action__invite: "\uCD08\uB300",
      action__search: "\uAC80\uC0C9",
      activeMembersTab: {
        menuAction__remove: "\uD68C\uC6D0 \uC81C\uAC70",
        tableHeader__actions: "\uC791\uC5C5",
        tableHeader__joined: "\uAC00\uC785\uB428",
        tableHeader__role: "\uC5ED\uD560",
        tableHeader__user: "\uC0AC\uC6A9\uC790"
      },
      detailsTitle__emptyRow: "\uD45C\uC2DC\uD560 \uBA64\uBC84 \uC5C6\uC74C",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "\uC774\uBA54\uC77C \uB3C4\uBA54\uC778\uC744 \uC870\uC9C1\uACFC \uC5F0\uACB0\uD574 \uC0AC\uC6A9\uC790\uB97C \uCD08\uB300\uD558\uC138\uC694. \uD574\uB2F9 \uB3C4\uBA54\uC778\uC73C\uB85C \uAC00\uC785\uD55C \uC0AC\uC6A9\uC790\uB294 \uC5B8\uC81C\uB4E0 \uC870\uC9C1\uC5D0 \uCC38\uC5EC\uD560 \uC218 \uC788\uC5B4\uC694.",
          headerTitle: "\uC790\uB3D9 \uCD08\uB300",
          primaryButton: "\uC778\uC99D\uB41C \uB3C4\uBA54\uC778 \uAD00\uB9AC"
        },
        table__emptyRow: "\uD45C\uC2DC\uD560 \uCD08\uB300\uAC00 \uC5C6\uC5B4\uC694"
      },
      invitedMembersTab: {
        menuAction__revoke: "\uCD08\uB300 \uCDE8\uC18C",
        tableHeader__invited: "\uCD08\uB300\uB428"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "\uD574\uB2F9 \uB3C4\uBA54\uC778\uC73C\uB85C \uAC00\uC785\uD55C \uC0AC\uC6A9\uC790\uB294 \uC870\uC9C1 \uCC38\uC5EC \uC694\uCCAD\uC744 \uC81C\uC548\uBC1B\uC744 \uC218 \uC788\uC5B4\uC694.",
          headerTitle: "\uC790\uB3D9 \uC81C\uC548",
          primaryButton: "\uC778\uC99D\uB41C \uB3C4\uBA54\uC778 \uAD00\uB9AC"
        },
        menuAction__approve: "\uC2B9\uC778",
        menuAction__reject: "\uAC70\uC808",
        tableHeader__requested: "\uC694\uCCAD\uB428",
        table__emptyRow: "\uC694\uCCAD\uC774 \uC5C6\uC5B4\uC694"
      },
      start: {
        headerTitle__invitations: "\uCD08\uB300",
        headerTitle__members: "\uBA64\uBC84",
        headerTitle__requests: "\uC694\uCCAD"
      },
      alerts: {
        roleSetMigrationInProgress: {
          title: "\uC5ED\uD560\uC774 \uC7A0\uC2DC \uC7A0\uACBC\uC5B4\uC694",
          subtitle: "\uC0AC\uC6A9 \uAC00\uB2A5\uD55C \uC5ED\uD560\uC744 \uC5C5\uB370\uC774\uD2B8\uD558\uACE0 \uC788\uC5B4\uC694. \uB05D\uB098\uBA74 \uB2E4\uC2DC \uC5ED\uD560\uC744 \uBC14\uAFC0 \uC218 \uC788\uC5B4\uC694."
        }
      }
    },
    navbar: {
      apiKeys: "API \uD0A4",
      billing: "\uACB0\uC81C",
      description: "\uC870\uC9C1\uC744 \uAD00\uB9AC\uD558\uC138\uC694.",
      general: "\uC77C\uBC18",
      members: "\uBA64\uBC84",
      title: "\uC870\uC9C1"
    },
    plansPage: {
      alerts: {
        noPermissionsToManageBilling: "\uC774 \uC870\uC9C1\uC758 \uACB0\uC81C\uB97C \uAD00\uB9AC\uD560 \uAD8C\uD55C\uC774 \uC5C6\uC5B4\uC694."
      },
      title: "\uD50C\uB79C"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: "\uACC4\uC18D\uD558\uB824\uBA74 \uC544\uB798\uC5D0 {{organizationName}}\uC744(\uB97C) \uC785\uB825\uD558\uC138\uC694.",
          messageLine1: "\uC774 \uC870\uC9C1\uC744 \uC0AD\uC81C\uD560\uAE4C\uC694?",
          messageLine2: "\uC774 \uC791\uC5C5\uC740 \uB418\uB3CC\uB9B4 \uC218 \uC5C6\uC5B4\uC694.",
          successMessage: "\uC870\uC9C1\uC744 \uC0AD\uC81C\uD588\uC5B4\uC694.",
          title: "\uC870\uC9C1 \uC0AD\uC81C"
        },
        leaveOrganization: {
          actionDescription: '\uACC4\uC18D\uD558\uB824\uBA74 \uC544\uB798\uC5D0 "{{organizationName}}"\uC744(\uB97C) \uC785\uB825\uD558\uC138\uC694.',
          messageLine1: "\uC774 \uC870\uC9C1\uC744 \uD0C8\uD1F4\uD560\uAE4C\uC694? \uC870\uC9C1\uACFC \uAD00\uB828 \uC571\uC5D0 \uB300\uD55C \uC811\uADFC \uAD8C\uD55C\uC744 \uC783\uAC8C \uB3FC\uC694.",
          messageLine2: "\uC774 \uC791\uC5C5\uC740 \uB418\uB3CC\uB9B4 \uC218 \uC5C6\uC5B4\uC694.",
          successMessage: "\uC870\uC9C1\uC744 \uD0C8\uD1F4\uD588\uC5B4\uC694.",
          title: "\uC870\uC9C1 \uB5A0\uB098\uAE30"
        },
        title: "\uC704\uD5D8"
      },
      domainSection: {
        menuAction__manage: "\uAD00\uB9AC",
        menuAction__remove: "\uC0AD\uC81C",
        menuAction__verify: "\uC778\uC99D",
        primaryButton: "\uB3C4\uBA54\uC778 \uCD94\uAC00",
        subtitle: "\uC778\uC99D\uB41C \uC774\uBA54\uC77C \uB3C4\uBA54\uC778\uC744 \uAE30\uC900\uC73C\uB85C \uC0AC\uC6A9\uC790\uAC00 \uC790\uB3D9 \uCC38\uC5EC\uD558\uAC70\uB098 \uCC38\uC5EC \uC694\uCCAD\uC744 \uD560 \uC218 \uC788\uAC8C \uD574\uC694.",
        title: "\uC778\uC99D\uB41C \uB3C4\uBA54\uC778"
      },
      successMessage: "\uC870\uC9C1\uC774 \uC5C5\uB370\uC774\uD2B8\uB410\uC5B4\uC694.",
      title: "\uD504\uB85C\uD544 \uC5C5\uB370\uC774\uD2B8"
    },
    removeDomainPage: {
      messageLine1: "\uC774\uBA54\uC77C \uB3C4\uBA54\uC778 {{domain}}\uC774(\uAC00) \uC0AD\uC81C\uB3FC\uC694.",
      messageLine2: "\uC774\uD6C4\uC5D0\uB294 \uC0AC\uC6A9\uC790\uAC00 \uC870\uC9C1\uC5D0 \uC790\uB3D9\uC73C\uB85C \uCC38\uC5EC\uD560 \uC218 \uC5C6\uC5B4\uC694.",
      successMessage: "{{domain}}\uC774(\uAC00) \uC0AD\uC81C\uB410\uC5B4\uC694.",
      title: "\uB3C4\uBA54\uC778 \uC0AD\uC81C"
    },
    start: {
      headerTitle__general: "\uC77C\uBC18",
      headerTitle__members: "\uBA64\uBC84",
      profileSection: {
        primaryButton: "\uD504\uB85C\uD544 \uC5C5\uB370\uC774\uD2B8",
        title: "\uC870\uC9C1 \uD504\uB85C\uD544",
        uploadAction__title: "\uB85C\uACE0"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "\uC774 \uB3C4\uBA54\uC778\uC744 \uC0AD\uC81C\uD558\uBA74 \uCD08\uB300\uB41C \uC0AC\uC6A9\uC790\uC5D0\uAC8C \uC601\uD5A5\uC774 \uC788\uC5B4\uC694.",
        removeDomainActionLabel__remove: "\uB3C4\uBA54\uC778 \uC0AD\uC81C",
        removeDomainSubtitle: "\uC774 \uB3C4\uBA54\uC778\uC744 \uC778\uC99D\uB41C \uB3C4\uBA54\uC778 \uBAA9\uB85D\uC5D0\uC11C \uC0AD\uC81C",
        removeDomainTitle: "\uB3C4\uBA54\uC778 \uC0AD\uC81C"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "\uC0AC\uC6A9\uC790\uAC00 \uAC00\uC785\uD558\uBA74 \uC790\uB3D9\uC73C\uB85C \uCD08\uB300\uB418\uACE0 \uC5B8\uC81C\uB4E0\uC9C0 \uC870\uC9C1\uC5D0 \uCC38\uC5EC\uD560 \uC218 \uC788\uC5B4\uC694.",
        automaticInvitationOption__label: "\uC790\uB3D9 \uCD08\uB300",
        automaticSuggestionOption__description: "\uC0AC\uC6A9\uC790\uB294 \uCC38\uC5EC \uC694\uCCAD \uC81C\uC548\uC744 \uBC1B\uC9C0\uB9CC, \uAD00\uB9AC\uC790\uAC00 \uC2B9\uC778\uD574\uC57C \uC870\uC9C1\uC5D0 \uCC38\uC5EC\uD560 \uC218 \uC788\uC5B4\uC694.",
        automaticSuggestionOption__label: "\uC790\uB3D9 \uC81C\uC548",
        calloutInfoLabel: "\uB4F1\uB85D \uBC29\uC2DD \uBCC0\uACBD\uC740 \uC0C8 \uC0AC\uC6A9\uC790\uC5D0\uAC8C\uB9CC \uC801\uC6A9\uB3FC\uC694.",
        calloutInvitationCountLabel: "\uB300\uAE30 \uC911\uC778 \uCD08\uB300: {{count}}",
        calloutSuggestionCountLabel: "\uB300\uAE30 \uC911\uC778 \uC81C\uC548: {{count}}",
        manualInvitationOption__description: "\uC0AC\uC6A9\uC790\uB294 \uC218\uB3D9\uC73C\uB85C\uB9CC \uC870\uC9C1\uC5D0 \uCD08\uB300\uD560 \uC218 \uC788\uC5B4\uC694.",
        manualInvitationOption__label: "\uC790\uB3D9 \uB4F1\uB85D \uC5C6\uC74C",
        subtitle: "\uC774 \uB3C4\uBA54\uC778 \uC0AC\uC6A9\uC790\uAC00 \uC870\uC9C1\uC5D0 \uCC38\uC5EC\uD558\uB294 \uBC29\uBC95\uC744 \uC120\uD0DD\uD558\uC138\uC694."
      },
      start: {
        headerTitle__danger: "\uC704\uD5D8",
        headerTitle__enrollment: "\uB4F1\uB85D \uC635\uC158"
      },
      subtitle: "\uB3C4\uBA54\uC778 {{domain}}\uC774(\uAC00) \uC778\uC99D\uB410\uC5B4\uC694. \uB4F1\uB85D \uBC29\uC2DD\uC744 \uC120\uD0DD\uD574 \uC8FC\uC138\uC694.",
      title: "{{domain}} \uC5C5\uB370\uC774\uD2B8"
    },
    verifyDomainPage: {
      formSubtitle: "\uC774\uBA54\uC77C\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC\uB97C \uBC1B\uC9C0 \uBABB\uD558\uC168\uB098\uC694? \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "\uB3C4\uBA54\uC778 {{domainName}}\uC740 \uC774\uBA54\uC77C\uB85C \uC778\uC99D\uC774 \uD544\uC694\uD574\uC694.",
      subtitleVerificationCodeScreen: "{{emailAddress}}\uB85C \uC778\uC99D \uCF54\uB4DC\uAC00 \uC804\uC1A1\uB410\uC5B4\uC694. \uCF54\uB4DC\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694.",
      title: "\uB3C4\uBA54\uC778 \uC778\uC99D"
    }
  },
  organizationSwitcher: {
    action__closeOrganizationSwitcher: "\uC870\uC9C1 \uC804\uD658\uAE30 \uB2EB\uAE30",
    action__createOrganization: "\uC870\uC9C1 \uB9CC\uB4E4\uAE30",
    action__invitationAccept: "\uCC38\uC5EC",
    action__manageOrganization: "\uC870\uC9C1 \uAD00\uB9AC",
    action__openOrganizationSwitcher: "\uC870\uC9C1 \uC804\uD658\uAE30 \uC5F4\uAE30",
    action__suggestionsAccept: "\uCC38\uC5EC \uC694\uCCAD",
    notSelected: "\uC120\uD0DD\uD55C \uC870\uC9C1 \uC5C6\uC74C",
    personalWorkspace: "\uAC1C\uC778 \uC6CC\uD06C\uC2A4\uD398\uC774\uC2A4",
    suggestionsAcceptedLabel: "\uC2B9\uC778 \uB300\uAE30"
  },
  paginationButton__next: "\uB2E4\uC74C",
  paginationButton__previous: "\uC774\uC804",
  paginationRowText__displaying: "\uD45C\uC2DC\uC911",
  paginationRowText__of: "\uC758",
  reverification: {
    alternativeMethods: {
      actionLink: "\uB3C4\uC6C0 \uC694\uCCAD\uD558\uAE30",
      actionText: "\uC774 \uBC29\uBC95\uC774 \uC5C6\uB098\uC694?",
      blockButton__backupCode: "\uBC31\uC5C5 \uCF54\uB4DC \uC0AC\uC6A9",
      blockButton__emailCode: "{{identifier}}\uB85C \uC774\uBA54\uC77C \uCF54\uB4DC \uBCF4\uB0B4\uAE30",
      blockButton__passkey: "\uD328\uC2A4\uD0A4 \uC0AC\uC6A9\uD558\uAE30",
      blockButton__password: "\uBE44\uBC00\uBC88\uD638\uB85C \uACC4\uC18D",
      blockButton__phoneCode: "{{identifier}}\uB85C SMS \uCF54\uB4DC \uBCF4\uB0B4\uAE30",
      blockButton__totp: "\uC778\uC99D \uC571 \uC0AC\uC6A9\uD558\uAE30",
      getHelp: {
        blockButton__emailSupport: "\uC774\uBA54\uC77C \uC9C0\uC6D0",
        content: "\uACC4\uC815 \uC778\uC99D\uC5D0 \uBB38\uC81C\uAC00 \uC788\uB2E4\uBA74 \uC774\uBA54\uC77C\uB85C \uBB38\uC758\uD574 \uC8FC\uC138\uC694. \uCD5C\uB300\uD55C \uBE60\uB974\uAC8C \uB3C4\uC640\uB4DC\uB9B4\uAC8C\uC694.",
        title: "\uB3C4\uC6C0 \uC694\uCCAD\uD558\uAE30"
      },
      subtitle: "\uBB38\uC81C\uAC00 \uC788\uB098\uC694? \uB2E4\uB978 \uBC29\uBC95\uC73C\uB85C \uC778\uC99D\uD560 \uC218 \uC788\uC5B4\uC694.",
      title: "\uB2E4\uB978 \uBC29\uBC95 \uC0AC\uC6A9\uD558\uAE30"
    },
    backupCodeMfa: {
      subtitle: "2\uB2E8\uACC4 \uC778\uC99D \uC124\uC815 \uB54C \uBC1B\uC740 \uBC31\uC5C5 \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      title: "\uBC31\uC5C5 \uCF54\uB4DC \uC785\uB825"
    },
    emailCode: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC\uB97C \uBC1B\uC9C0 \uBABB\uD558\uC168\uB098\uC694? \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "\uC774\uBA54\uC77C\uB85C \uC804\uC1A1\uB41C \uCF54\uB4DC\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694",
      title: "\uC778\uC99D\uC774 \uD544\uC694\uD574\uC694"
    },
    noAvailableMethods: {
      message: "\uC778\uC99D\uC744 \uC9C4\uD589\uD560 \uC218 \uC5C6\uC5B4\uC694. \uC124\uC815\uB41C \uC778\uC99D \uC218\uB2E8\uC774 \uC5C6\uC5B4\uC694.",
      subtitle: "\uC624\uB958\uAC00 \uBC1C\uC0DD\uD588\uC5B4\uC694",
      title: "\uACC4\uC815\uC744 \uD655\uC778\uD560 \uC218 \uC5C6\uC5B4\uC694"
    },
    passkey: {
      blockButton__passkey: "\uD328\uC2A4\uD0A4 \uC0AC\uC6A9\uD558\uAE30",
      subtitle: "\uD328\uC2A4\uD0A4\uB85C \uC2E0\uC6D0\uC744 \uD655\uC778\uD574\uC694. \uAE30\uAE30\uC5D0\uC11C \uC9C0\uBB38, \uC5BC\uAD74 \uB610\uB294 \uD654\uBA74 \uC7A0\uAE08\uC744 \uC694\uCCAD\uD560 \uC218 \uC788\uC5B4\uC694.",
      title: "\uD328\uC2A4\uD0A4 \uC0AC\uC6A9"
    },
    password: {
      actionLink: "\uB2E4\uB978 \uBC29\uBC95 \uC0AC\uC6A9\uD558\uAE30",
      subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uD604\uC7AC \uBE44\uBC00\uBC88\uD638\uB97C \uC785\uB825\uD558\uC138\uC694",
      title: "\uC778\uC99D\uC774 \uD544\uC694\uD574\uC694"
    },
    phoneCode: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC\uB97C \uBC1B\uC9C0 \uBABB\uD558\uC168\uB098\uC694? \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "\uD734\uB300\uD3F0\uC73C\uB85C \uC804\uC1A1\uB41C \uCF54\uB4DC\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694",
      title: "\uC778\uC99D\uC774 \uD544\uC694\uD574\uC694"
    },
    phoneCodeMfa: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC\uB97C \uBC1B\uC9C0 \uBABB\uD558\uC168\uB098\uC694? \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "\uD734\uB300\uD3F0\uC73C\uB85C \uC804\uC1A1\uB41C \uCF54\uB4DC\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694",
      title: "\uC778\uC99D\uC774 \uD544\uC694\uD574\uC694"
    },
    totpMfa: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      subtitle: "\uC778\uC99D \uC571\uC5D0\uC11C \uC0DD\uC131\uB41C \uCF54\uB4DC\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694",
      title: "\uC778\uC99D\uC774 \uD544\uC694\uD574\uC694"
    }
  },
  signIn: {
    accountSwitcher: {
      action__addAccount: "\uACC4\uC815 \uCD94\uAC00",
      action__signOutAll: "\uBAA8\uB4E0 \uACC4\uC815 \uB85C\uADF8\uC544\uC6C3",
      subtitle: "\uACC4\uC18D\uD560 \uACC4\uC815\uC744 \uC120\uD0DD\uD574 \uC8FC\uC138\uC694.",
      title: "\uACC4\uC815 \uC120\uD0DD"
    },
    alternativeMethods: {
      actionLink: "\uB3C4\uC6C0 \uC694\uCCAD\uD558\uAE30",
      actionText: "\uC774 \uBC29\uBC95\uC774 \uC5C6\uB098\uC694?",
      blockButton__backupCode: "\uBC31\uC5C5 \uCF54\uB4DC \uC0AC\uC6A9\uD558\uAE30",
      blockButton__emailCode: "{{identifier}}\uB85C \uC774\uBA54\uC77C \uCF54\uB4DC \uBCF4\uB0B4\uAE30",
      blockButton__emailLink: "{{identifier}}\uB85C \uC774\uBA54\uC77C \uB9C1\uD06C \uBCF4\uB0B4\uAE30",
      blockButton__passkey: "\uD328\uC2A4\uD0A4\uB85C \uB85C\uADF8\uC778",
      blockButton__password: "\uBE44\uBC00\uBC88\uD638\uB85C \uB85C\uADF8\uC778",
      blockButton__phoneCode: "{{identifier}}\uB85C SMS \uCF54\uB4DC \uBCF4\uB0B4\uAE30",
      blockButton__totp: "\uC778\uC99D \uC571 \uC0AC\uC6A9\uD558\uAE30",
      getHelp: {
        blockButton__emailSupport: "\uC774\uBA54\uC77C \uC9C0\uC6D0",
        content: "\uACC4\uC815 \uB85C\uADF8\uC778\uC5D0 \uBB38\uC81C\uAC00 \uC788\uB2E4\uBA74 \uC774\uBA54\uC77C\uB85C \uC54C\uB824 \uC8FC\uC138\uC694. \uCD5C\uB300\uD55C \uBE60\uB974\uAC8C \uC811\uADFC\uC744 \uBCF5\uAD6C\uD574 \uB4DC\uB9B4\uAC8C\uC694.",
        title: "\uB3C4\uC6C0 \uC694\uCCAD\uD558\uAE30"
      },
      subtitle: "\uBB38\uC81C\uAC00 \uC788\uB098\uC694? \uB2E4\uB978 \uBC29\uBC95\uC73C\uB85C \uB85C\uADF8\uC778\uD560 \uC218 \uC788\uC5B4\uC694.",
      title: "\uB2E4\uB978 \uBC29\uBC95 \uC0AC\uC6A9\uD558\uAE30"
    },
    alternativePhoneCodeProvider: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC\uB97C \uBC1B\uC9C0 \uBABB\uD558\uC168\uB098\uC694? \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "{{applicationName}}\uB85C \uACC4\uC18D\uD558\uB824\uBA74",
      title: "{{provider}} \uD655\uC778"
    },
    backupCodeMfa: {
      subtitle: "\uBC31\uC5C5 \uCF54\uB4DC\uB294 2\uB2E8\uACC4 \uC778\uC99D\uC744 \uC124\uC815\uD560 \uB54C \uBC1B\uC740 \uCF54\uB4DC\uC608\uC694",
      title: "\uBC31\uC5C5 \uCF54\uB4DC \uC785\uB825"
    },
    emailCode: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC \uC7AC\uC804\uC1A1",
      subtitle: "{{applicationName}}\uB85C \uACC4\uC18D\uD558\uB824\uBA74",
      title: "\uC774\uBA54\uC77C\uC744 \uD655\uC778\uD558\uC138\uC694"
    },
    emailCodeMfa: {
      formTitle: "\uC774\uBA54\uC77C\uC744 \uD655\uC778\uD558\uC138\uC694",
      resendButton: "\uCF54\uB4DC\uB97C \uBC1B\uC9C0 \uBABB\uD558\uC168\uB098\uC694? \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "{{applicationName}}\uB85C \uACC4\uC18D\uD558\uB824\uBA74",
      title: "\uC774\uBA54\uC77C\uC744 \uD655\uC778\uD558\uC138\uC694"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uB85C\uADF8\uC778 \uC2DC\uC791\uD55C \uAE30\uAE30\uC640 \uBE0C\uB77C\uC6B0\uC800\uC5D0\uC11C \uB9C1\uD06C\uB97C \uC5F4\uC5B4 \uC8FC\uC138\uC694.",
        title: "\uC774 \uAE30\uAE30\uC5D0\uC11C\uB294 \uC778\uC99D \uB9C1\uD06C\uB97C \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uC5B4\uC694"
      },
      expired: {
        subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uC6D0\uB798 \uD0ED\uC73C\uB85C \uB3CC\uC544\uAC00\uC138\uC694.",
        title: "\uC774 \uC778\uC99D \uB9C1\uD06C\uB294 \uB9CC\uB8CC\uB410\uC5B4\uC694"
      },
      failed: {
        subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uC6D0\uB798 \uD0ED\uC73C\uB85C \uB3CC\uC544\uAC00\uC138\uC694.",
        title: "\uC774 \uC778\uC99D \uB9C1\uD06C\uB294 \uC720\uD6A8\uD558\uC9C0 \uC54A\uC544\uC694"
      },
      formSubtitle: "\uC774\uBA54\uC77C\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uB9C1\uD06C\uB97C \uC0AC\uC6A9\uD558\uC138\uC694",
      formTitle: "\uC778\uC99D \uB9C1\uD06C",
      loading: {
        subtitle: "\uACE7 \uC774\uB3D9\uB3FC\uC694",
        title: "\uB85C\uADF8\uC778 \uC911..."
      },
      resendButton: "\uB9C1\uD06C \uC7AC\uC804\uC1A1",
      subtitle: "{{applicationName}}\uB85C \uACC4\uC18D\uD558\uB824\uBA74",
      title: "\uC774\uBA54\uC77C\uC744 \uD655\uC778\uD558\uC138\uC694",
      unusedTab: {
        title: "\uC774 \uD0ED\uC740 \uB2EB\uC544\uB3C4 \uB3FC\uC694"
      },
      verified: {
        subtitle: "\uACE7 \uC774\uB3D9\uB3FC\uC694",
        title: "\uB85C\uADF8\uC778\uC774 \uC644\uB8CC\uB410\uC5B4\uC694"
      },
      verifiedSwitchTab: {
        subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uC6D0\uB798 \uD0ED\uC73C\uB85C \uB3CC\uC544\uAC00\uC138\uC694",
        subtitleNewTab: "\uACC4\uC18D\uD558\uB824\uBA74 \uC0C8\uB85C \uC5F0 \uD0ED\uC73C\uB85C \uB3CC\uC544\uAC00\uC138\uC694",
        titleNewTab: "\uB2E4\uB978 \uD0ED\uC5D0\uC11C \uB85C\uADF8\uC778"
      }
    },
    emailLinkMfa: {
      formSubtitle: "\uC774\uBA54\uC77C\uB85C \uC804\uC1A1\uB41C \uD655\uC778 \uB9C1\uD06C\uB97C \uC0AC\uC6A9\uD558\uC138\uC694",
      resendButton: "\uB9C1\uD06C\uB97C \uBC1B\uC9C0 \uBABB\uD558\uC168\uB098\uC694? \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "{{applicationName}}(\uC73C)\uB85C \uACC4\uC18D",
      title: "\uC774\uBA54\uC77C\uC744 \uD655\uC778\uD558\uC138\uC694"
    },
    enterpriseConnections: {
      subtitle: "\uACC4\uC18D\uD560 \uAE30\uC5C5 \uACC4\uC815\uC744 \uC120\uD0DD\uD574 \uC8FC\uC138\uC694.",
      title: "\uAE30\uC5C5 \uACC4\uC815 \uC120\uD0DD"
    },
    forgotPassword: {
      formTitle: "\uBE44\uBC00\uBC88\uD638 \uC7AC\uC124\uC815 \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC \uC7AC\uC804\uC1A1",
      subtitle: "\uBE44\uBC00\uBC88\uD638\uB97C \uC7AC\uC124\uC815\uD558\uB824\uBA74",
      subtitle_email: "\uBA3C\uC800 \uC774\uBA54\uC77C\uB85C \uC804\uC1A1\uB41C \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      subtitle_phone: "\uBA3C\uC800 \uD734\uB300\uD3F0\uC73C\uB85C \uC804\uC1A1\uB41C \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      title: "\uBE44\uBC00\uBC88\uD638 \uC7AC\uC124\uC815"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "\uBE44\uBC00\uBC88\uD638 \uC7AC\uC124\uC815",
      label__alternativeMethods: "\uB610\uB294 \uB2E4\uB978 \uBC29\uBC95\uC73C\uB85C \uB85C\uADF8\uC778",
      title: "\uBE44\uBC00\uBC88\uD638\uB97C \uC78A\uC73C\uC168\uB098\uC694?"
    },
    newDeviceVerificationNotice: "\uC0C8\uB85C\uC6B4 \uAE30\uAE30\uC5D0\uC11C \uB85C\uADF8\uC778 \uC911\uC774\uC5D0\uC694. \uACC4\uC815 \uBCF4\uC548\uC744 \uC704\uD574 \uD655\uC778\uC744 \uC694\uCCAD\uD558\uACE0 \uC788\uC5B4\uC694.",
    noAvailableMethods: {
      message: "\uB85C\uADF8\uC778\uC744 \uACC4\uC18D\uD560 \uC218 \uC5C6\uC5B4\uC694. \uC0AC\uC6A9\uD560 \uC218 \uC788\uB294 \uC778\uC99D \uBC29\uBC95\uC774 \uC5C6\uC5B4\uC694.",
      subtitle: "\uC624\uB958\uAC00 \uBC1C\uC0DD\uD588\uC5B4\uC694",
      title: "\uB85C\uADF8\uC778\uD560 \uC218 \uC5C6\uC5B4\uC694"
    },
    passkey: {
      subtitle: "\uD328\uC2A4\uD0A4\uB85C \uB85C\uADF8\uC778\uD558\uBA74 \uBCF8\uC778 \uD655\uC778\uC774 \uB3FC\uC694. \uAE30\uAE30\uC5D0\uC11C \uC9C0\uBB38, \uC5BC\uAD74 \uB610\uB294 \uD654\uBA74 \uC7A0\uAE08\uC744 \uC694\uCCAD\uD560 \uC218 \uC788\uC5B4\uC694.",
      title: "\uD328\uC2A4\uD0A4 \uC0AC\uC6A9"
    },
    password: {
      actionLink: "\uB2E4\uB978 \uBC29\uBC95 \uC0AC\uC6A9\uD558\uAE30",
      subtitle: "\uACC4\uC815\uC5D0 \uB4F1\uB85D\uB41C \uBE44\uBC00\uBC88\uD638\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694",
      title: "\uBE44\uBC00\uBC88\uD638\uB97C \uC785\uB825\uD558\uC138\uC694"
    },
    passwordCompromised: {
      title: "\uBE44\uBC00\uBC88\uD638\uAC00 \uC720\uCD9C\uB410\uC744 \uC218 \uC788\uC5B4\uC694"
    },
    passwordPwned: {
      title: "\uC720\uCD9C\uB41C \uBE44\uBC00\uBC88\uD638\uC608\uC694"
    },
    passwordUntrusted: {
      title: "\uC2E0\uB8B0\uD560 \uC218 \uC5C6\uB294 \uBE44\uBC00\uBC88\uD638\uC608\uC694"
    },
    phoneCode: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "{{applicationName}}\uB85C \uACC4\uC18D\uD558\uB824\uBA74",
      title: "\uD734\uB300\uD3F0 \uD655\uC778"
    },
    phoneCodeMfa: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uD734\uB300\uD3F0\uC73C\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      title: "\uD734\uB300\uD3F0 \uD655\uC778"
    },
    resetPassword: {
      formButtonPrimary: "\uBE44\uBC00\uBC88\uD638 \uC7AC\uC124\uC815",
      requiredMessage: "\uBCF4\uC548\uC744 \uC704\uD574 \uBE44\uBC00\uBC88\uD638\uB97C \uC7AC\uC124\uC815\uD574\uC57C \uD574\uC694.",
      successMessage: "\uBE44\uBC00\uBC88\uD638\uAC00 \uBCC0\uACBD\uB410\uC5B4\uC694. \uB85C\uADF8\uC778 \uC911\uC774\uC5D0\uC694. \uC7A0\uC2DC\uB9CC \uAE30\uB2E4\uB824 \uC8FC\uC138\uC694.",
      title: "\uBE44\uBC00\uBC88\uD638 \uC7AC\uC124\uC815"
    },
    resetPasswordMfa: {
      detailsLabel: "\uBE44\uBC00\uBC88\uD638\uB97C \uC7AC\uC124\uC815\uD558\uAE30 \uC804\uC5D0 \uC2E0\uC6D0\uC744 \uD655\uC778\uD574\uC57C \uD574\uC694."
    },
    start: {
      actionLink: "\uD68C\uC6D0\uAC00\uC785",
      actionLink__join_waitlist: "\uB300\uAE30 \uBAA9\uB85D \uCC38\uC5EC",
      actionLink__use_email: "\uC774\uBA54\uC77C \uC0AC\uC6A9\uD558\uAE30",
      actionLink__use_email_username: "\uC774\uBA54\uC77C \uB610\uB294 \uC544\uC774\uB514 \uC0AC\uC6A9\uD558\uAE30",
      actionLink__use_passkey: "\uD328\uC2A4\uD0A4 \uC0AC\uC6A9\uD558\uAE30",
      actionLink__use_phone: "\uD734\uB300\uD3F0 \uBC88\uD638 \uC0AC\uC6A9\uD558\uAE30",
      actionLink__use_username: "\uC544\uC774\uB514 \uC0AC\uC6A9\uD558\uAE30",
      actionText: "\uACC4\uC815\uC774 \uC5C6\uC73C\uC2E0\uAC00\uC694?",
      actionText__join_waitlist: "\uBBF8\uB9AC \uC0AC\uC6A9\uD574 \uBCF4\uACE0 \uC2F6\uC73C\uC2E0\uAC00\uC694?",
      alternativePhoneCodeProvider: {
        actionLink: "\uB2E4\uB978 \uBC29\uBC95 \uC0AC\uC6A9\uD558\uAE30",
        label: "{{provider}} \uD734\uB300\uD3F0 \uBC88\uD638",
        subtitle: "{{provider}}\uB85C \uC778\uC99D \uCF54\uB4DC\uB97C \uBC1B\uC744 \uD734\uB300\uD3F0 \uBC88\uD638\uB97C \uC785\uB825\uD558\uC138\uC694.",
        title: "{{provider}}\uB85C {{applicationName}}\uC5D0 \uB85C\uADF8\uC778"
      },
      subtitle: "\uD658\uC601\uD574\uC694! \uACC4\uC18D\uD558\uB824\uBA74 \uB85C\uADF8\uC778\uD574 \uC8FC\uC138\uC694",
      subtitleCombined: void 0,
      title: "{{applicationName}}\uC5D0 \uB85C\uADF8\uC778",
      titleCombined: "{{applicationName}}\uB85C \uACC4\uC18D"
    },
    totpMfa: {
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uC778\uC99D \uC571\uC5D0\uC11C \uC0DD\uC131\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      title: "2\uB2E8\uACC4 \uC778\uC99D"
    },
    web3Solana: {
      subtitle: "\uB85C\uADF8\uC778\uD558\uB824\uBA74 \uC544\uB798\uC5D0\uC11C \uC9C0\uAC11\uC744 \uC120\uD0DD\uD558\uC138\uC694",
      title: "Solana\uB85C \uB85C\uADF8\uC778"
    }
  },
  signInEnterPasswordTitle: "\uBE44\uBC00\uBC88\uD638\uB97C \uC785\uB825\uD558\uC138\uC694",
  signUp: {
    alternativePhoneCodeProvider: {
      resendButton: "\uCF54\uB4DC\uB97C \uBC1B\uC9C0 \uBABB\uD558\uC168\uB098\uC694? \uB2E4\uC2DC \uBCF4\uB0B4\uAE30",
      subtitle: "{{provider}}\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      title: "{{provider}} \uC778\uC99D"
    },
    continue: {
      actionLink: "\uB85C\uADF8\uC778",
      actionText: "\uACC4\uC815\uC774 \uC788\uC73C\uC2E0\uAC00\uC694?",
      subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uB204\uB77D\uB41C \uD544\uB4DC\uC5D0 \uAC12\uC744 \uC785\uB825\uD558\uC138\uC694",
      title: "\uB204\uB77D\uB41C \uD544\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694"
    },
    emailCode: {
      formSubtitle: "\uC774\uBA54\uC77C \uC8FC\uC18C\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC \uC7AC\uC804\uC1A1",
      subtitle: "\uC774\uBA54\uC77C\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      title: "\uC774\uBA54\uC77C \uC778\uC99D"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uAC00\uC785\uC744 \uC2DC\uC791\uD55C \uAE30\uAE30\uC640 \uBE0C\uB77C\uC6B0\uC800\uC5D0\uC11C \uB9C1\uD06C\uB97C \uC5F4\uC5B4 \uC8FC\uC138\uC694.",
        title: "\uC774 \uAE30\uAE30\uC5D0\uC11C\uB294 \uC778\uC99D \uB9C1\uD06C\uB97C \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uC5B4\uC694"
      },
      formSubtitle: "\uC774\uBA54\uC77C\uB85C \uBC1B\uC740 \uC778\uC99D \uB9C1\uD06C\uB97C \uC0AC\uC6A9\uD574 \uC8FC\uC138\uC694.",
      formTitle: "\uC778\uC99D \uB9C1\uD06C",
      loading: {
        title: "\uAC00\uC785 \uC911..."
      },
      resendButton: "\uB9C1\uD06C \uC7AC\uC804\uC1A1",
      subtitle: "{{applicationName}}\uB85C \uACC4\uC18D\uD558\uB824\uBA74",
      title: "\uC774\uBA54\uC77C \uC778\uC99D\uD558\uAE30",
      verified: {
        title: "\uAC00\uC785\uC774 \uC644\uB8CC\uB410\uC5B4\uC694"
      },
      verifiedSwitchTab: {
        subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uC0C8\uB85C \uC5F0 \uD0ED\uC73C\uB85C \uB3CC\uC544\uAC00\uAE30",
        subtitleNewTab: "\uACC4\uC18D\uD558\uB824\uBA74 \uC774\uC804 \uD0ED\uC73C\uB85C \uB3CC\uC544\uAC00\uAE30",
        title: "\uC774\uBA54\uC77C \uC778\uC99D \uC131\uACF5"
      }
    },
    enterpriseConnections: {
      subtitle: "\uACC4\uC18D\uD560 \uAE30\uC5C5 \uACC4\uC815\uC744 \uC120\uD0DD\uD574 \uC8FC\uC138\uC694.",
      title: "\uAE30\uC5C5 \uACC4\uC815 \uC120\uD0DD"
    },
    legalConsent: {
      checkbox: {
        label__onlyPrivacyPolicy: '{{ privacyPolicyLink || link("\uAC1C\uC778\uC815\uBCF4\uCC98\uB9AC\uBC29\uCE68") }}\uC5D0 \uB3D9\uC758\uD574\uC694',
        label__onlyTermsOfService: '{{ termsOfServiceLink || link("\uC11C\uBE44\uC2A4 \uC57D\uAD00") }}\uC5D0 \uB3D9\uC758\uD574\uC694',
        label__termsOfServiceAndPrivacyPolicy: '{{ termsOfServiceLink || link("\uC11C\uBE44\uC2A4 \uC57D\uAD00") }}\uACFC {{ privacyPolicyLink || link("\uAC1C\uC778\uC815\uBCF4\uCC98\uB9AC\uBC29\uCE68") }}\uC5D0 \uB3D9\uC758\uD574\uC694'
      },
      continue: {
        subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uC57D\uAD00\uC5D0 \uB3D9\uC758\uD574 \uC8FC\uC138\uC694",
        title: "\uC57D\uAD00 \uB3D9\uC758"
      }
    },
    phoneCode: {
      formSubtitle: "\uD734\uB300\uD3F0 \uBC88\uD638\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694.",
      formTitle: "\uC778\uC99D \uCF54\uB4DC",
      resendButton: "\uCF54\uB4DC \uC7AC\uC804\uC1A1",
      subtitle: "\uD734\uB300\uD3F0\uC73C\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      title: "\uD734\uB300\uD3F0 \uBC88\uD638 \uC778\uC99D"
    },
    restrictedAccess: {
      actionLink: "\uB85C\uADF8\uC778",
      actionText: "\uC774\uBBF8 \uACC4\uC815\uC774 \uC788\uB098\uC694?",
      blockButton__emailSupport: "\uC774\uBA54\uC77C \uC9C0\uC6D0",
      blockButton__joinWaitlist: "\uB300\uAE30 \uBAA9\uB85D \uCC38\uC5EC",
      subtitle: "\uD604\uC7AC \uD68C\uC6D0\uAC00\uC785\uC774 \uC911\uB2E8\uB3FC \uC788\uC5B4\uC694. \uC811\uADFC \uAD8C\uD55C\uC774 \uD544\uC694\uD558\uB2E4\uBA74 \uC9C0\uC6D0\uD300\uC5D0 \uBB38\uC758\uD574 \uC8FC\uC138\uC694.",
      subtitleWaitlist: "\uD68C\uC6D0\uAC00\uC785\uC774 \uC911\uB2E8\uB3FC \uC788\uC5B4\uC694. \uCD9C\uC2DC \uC18C\uC2DD\uC744 \uAC00\uC7A5 \uBA3C\uC800 \uBC1B\uC73C\uB824\uBA74 \uB300\uAE30 \uBAA9\uB85D\uC5D0 \uCC38\uC5EC\uD574 \uC8FC\uC138\uC694.",
      title: "\uC811\uADFC \uC81C\uD55C"
    },
    start: {
      actionLink: "\uB85C\uADF8\uC778\uD558\uAE30",
      actionLink__use_email: "\uC774\uBA54\uC77C\uB85C \uAC00\uC785",
      actionLink__use_phone: "\uD734\uB300\uD3F0\uC73C\uB85C \uAC00\uC785",
      actionText: "\uACC4\uC815\uC774 \uC788\uC73C\uC2E0\uAC00\uC694?",
      alternativePhoneCodeProvider: {
        actionLink: "\uB2E4\uB978 \uBC29\uBC95 \uC0AC\uC6A9\uD558\uAE30",
        label: "{{provider}} \uD734\uB300\uD3F0 \uBC88\uD638",
        subtitle: "{{provider}}\uB85C \uC778\uC99D \uCF54\uB4DC\uB97C \uBC1B\uC744 \uD734\uB300\uD3F0 \uBC88\uD638\uB97C \uC785\uB825\uD558\uC138\uC694.",
        title: "{{provider}}\uB85C {{applicationName}}\uC5D0 \uAC00\uC785"
      },
      subtitle: "\uD658\uC601\uD574\uC694! \uC544\uB798 \uC815\uBCF4\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694.",
      subtitleCombined: "\uD658\uC601\uD574\uC694! \uC544\uB798 \uC815\uBCF4\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694.",
      title: "\uACC4\uC815 \uB9CC\uB4E4\uAE30",
      titleCombined: "\uACC4\uC815 \uB9CC\uB4E4\uAE30"
    },
    web3Solana: {
      subtitle: "\uAC00\uC785\uD558\uB824\uBA74 \uC544\uB798\uC5D0\uC11C \uC9C0\uAC11\uC744 \uC120\uD0DD\uD558\uC138\uC694",
      title: "Solana\uB85C \uAC00\uC785"
    }
  },
  socialButtonsBlockButton: "{{provider|titleize}}\uB85C \uACC4\uC18D\uD558\uAE30",
  socialButtonsBlockButtonManyInView: "{{provider|titleize}}",
  taskChooseOrganization: {
    chooseOrganization: {
      action__createOrganization: "\uC0C8 \uC870\uC9C1 \uB9CC\uB4E4\uAE30",
      action__invitationAccept: "\uCC38\uC5EC",
      action__suggestionsAccept: "\uCC38\uC5EC \uC694\uCCAD",
      subtitle: "\uAE30\uC874 \uC870\uC9C1\uC5D0 \uCC38\uC5EC\uD558\uAC70\uB098 \uC0C8 \uC870\uC9C1\uC744 \uB9CC\uB4DC\uC138\uC694",
      subtitle__createOrganizationDisabled: "\uAE30\uC874 \uC870\uC9C1\uC5D0 \uCC38\uC5EC\uD558\uC138\uC694",
      suggestionsAcceptedLabel: "\uC2B9\uC778 \uB300\uAE30 \uC911",
      title: "\uC870\uC9C1 \uC120\uD0DD"
    },
    createOrganization: {
      formButtonReset: "\uCDE8\uC18C",
      formButtonSubmit: "\uACC4\uC18D",
      formFieldInputPlaceholder__name: "\uB0B4 \uC870\uC9C1",
      formFieldInputPlaceholder__slug: "nae-jigug",
      formFieldLabel__name: "\uC774\uB984",
      formFieldLabel__slug: "\uC2AC\uB7EC\uADF8",
      subtitle: "\uC870\uC9C1 \uC815\uBCF4\uB97C \uC785\uB825\uD574 \uACC4\uC18D\uD558\uC138\uC694",
      title: "\uC870\uC9C1 \uC124\uC815"
    },
    organizationCreationDisabled: {
      title: "\uC870\uC9C1\uC5D0 \uC18C\uC18D\uB3FC\uC57C \uD574\uC694",
      subtitle: "\uCD08\uB300\uAC00 \uD544\uC694\uD558\uBA74 \uC870\uC9C1 \uAD00\uB9AC\uC790\uC5D0\uAC8C \uBB38\uC758\uD574 \uC8FC\uC138\uC694."
    },
    signOut: {
      actionLink: "\uB85C\uADF8\uC544\uC6C3",
      actionText: "{{identifier}}\uB85C \uB85C\uADF8\uC778\uB428"
    },
    alerts: {
      organizationAlreadyExists: "\uAC10\uC9C0\uB41C \uD68C\uC0AC \uC774\uB984({{organizationName}})\uACFC {{organizationDomain}}\uC5D0 \uD574\uB2F9\uD558\uB294 \uC870\uC9C1\uC774 \uC774\uBBF8 \uC788\uC5B4\uC694. \uCD08\uB300\uB85C \uCC38\uC5EC\uD574 \uC8FC\uC138\uC694."
    }
  },
  taskResetPassword: {
    formButtonPrimary: "\uBE44\uBC00\uBC88\uD638 \uC7AC\uC124\uC815",
    signOut: {
      actionLink: "\uB85C\uADF8\uC544\uC6C3",
      actionText: "{{identifier}}\uB85C \uB85C\uADF8\uC778\uB428"
    },
    subtitle: "\uACC4\uC18D\uD558\uB824\uBA74 \uBE44\uBC00\uBC88\uD638\uB97C \uC0C8\uB85C \uC124\uC815\uD574\uC57C \uD574\uC694",
    title: "\uBE44\uBC00\uBC88\uD638 \uC7AC\uC124\uC815"
  },
  unstable__errors: {
    already_a_member_in_organization: "{{email}}\uC740(\uB294) \uC774\uBBF8 \uC774 \uC870\uC9C1\uC758 \uBA64\uBC84\uC608\uC694.",
    avatar_file_size_exceeded: "\uD30C\uC77C \uD06C\uAE30\uAC00 \uCD5C\uB300 10MB \uC81C\uD55C\uC744 \uCD08\uACFC\uD574\uC694. \uB354 \uC791\uC740 \uD30C\uC77C\uC744 \uC120\uD0DD\uD574 \uC8FC\uC138\uC694.",
    avatar_file_type_invalid: "\uC9C0\uC6D0\uB418\uC9C0 \uC54A\uB294 \uD30C\uC77C \uD615\uC2DD\uC774\uC5D0\uC694. JPG, PNG, GIF \uB610\uB294 WEBP \uC774\uBBF8\uC9C0\uB97C \uC5C5\uB85C\uB4DC\uD574 \uC8FC\uC138\uC694.",
    captcha_invalid: "\uBCF4\uC548 \uAC80\uC99D\uC5D0 \uC2E4\uD328\uD574 \uD68C\uC6D0\uAC00\uC785\uC744 \uC644\uB8CC\uD560 \uC218 \uC5C6\uC5B4\uC694. \uD398\uC774\uC9C0\uB97C \uC0C8\uB85C\uACE0\uCE68\uD55C \uB4A4 \uB2E4\uC2DC \uC2DC\uB3C4\uD558\uAC70\uB098 \uC9C0\uC6D0\uD300\uC5D0 \uBB38\uC758\uD574 \uC8FC\uC138\uC694.",
    captcha_unavailable: "\uBD07 \uAC80\uC99D\uC5D0 \uC2E4\uD328\uD574 \uD68C\uC6D0\uAC00\uC785\uC744 \uC644\uB8CC\uD560 \uC218 \uC5C6\uC5B4\uC694. \uD398\uC774\uC9C0\uB97C \uC0C8\uB85C\uACE0\uCE68\uD55C \uB4A4 \uB2E4\uC2DC \uC2DC\uB3C4\uD558\uAC70\uB098 \uC9C0\uC6D0\uD300\uC5D0 \uBB38\uC758\uD574 \uC8FC\uC138\uC694.",
    form_code_incorrect: void 0,
    form_email_address_blocked: "\uC784\uC2DC \uC774\uBA54\uC77C \uC11C\uBE44\uC2A4\uB294 \uC9C0\uC6D0\uB418\uC9C0 \uC54A\uC544\uC694. \uACC4\uC815\uC744 \uB9CC\uB4E4\uB824\uBA74 \uC77C\uBC18 \uC774\uBA54\uC77C \uC8FC\uC18C\uB97C \uC0AC\uC6A9\uD574 \uC8FC\uC138\uC694.",
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: "\uC774 \uC815\uBCF4\uC640 \uC77C\uCE58\uD558\uB294 \uACC4\uC815\uC744 \uCC3E\uC744 \uC218 \uC5C6\uC5B4\uC694.",
    form_new_password_matches_current: "\uC0C8 \uBE44\uBC00\uBC88\uD638\uB294 \uD604\uC7AC \uBE44\uBC00\uBC88\uD638\uC640 \uAC19\uC744 \uC218 \uC5C6\uC5B4\uC694.",
    form_param_format_invalid: void 0,
    form_param_format_invalid__email_address: "\uC774\uBA54\uC77C \uC8FC\uC18C \uD615\uC2DD\uC774 \uC62C\uBC14\uB974\uC9C0 \uC54A\uC544\uC694.",
    form_param_format_invalid__phone_number: "\uC804\uD654\uBC88\uD638\uB294 \uC62C\uBC14\uB978 \uAD6D\uC81C \uD615\uC2DD\uC774\uC5B4\uC57C \uD574\uC694.",
    form_param_max_length_exceeded__first_name: "\uC774\uB984\uC740 256\uC790\uB97C \uB118\uC744 \uC218 \uC5C6\uC5B4\uC694.",
    form_param_max_length_exceeded__last_name: "\uC131\uC740 256\uC790\uB97C \uB118\uC744 \uC218 \uC5C6\uC5B4\uC694.",
    form_param_max_length_exceeded__name: "\uC774\uB984\uC740 256\uC790\uB97C \uB118\uC744 \uC218 \uC5C6\uC5B4\uC694.",
    form_param_nil: void 0,
    form_param_type_invalid: void 0,
    form_param_type_invalid__email_address: void 0,
    form_param_type_invalid__phone_number: void 0,
    form_param_value_invalid: void 0,
    form_password_incorrect: void 0,
    form_password_or_identifier_incorrect: "\uBE44\uBC00\uBC88\uD638 \uB610\uB294 \uC774\uBA54\uC77C \uC8FC\uC18C\uAC00 \uC62C\uBC14\uB974\uC9C0 \uC54A\uC544\uC694. \uB2E4\uC2DC \uC2DC\uB3C4\uD558\uAC70\uB098 \uB2E4\uB978 \uBC29\uBC95\uC744 \uC0AC\uC6A9\uD574 \uBCF4\uC138\uC694.",
    form_password_length_too_short: "\uBE44\uBC00\uBC88\uD638\uAC00 \uB108\uBB34 \uC9E7\uC544\uC694. \uCD5C\uC18C 8\uC790 \uC774\uC0C1\uC774\uC5B4\uC57C \uD574\uC694.",
    form_password_not_strong_enough: "\uBE44\uBC00\uBC88\uD638\uAC00 \uCDA9\uBD84\uD788 \uC548\uC804\uD558\uC9C0 \uC54A\uC544\uC694.",
    form_password_pwned: "\uC774 \uBE44\uBC00\uBC88\uD638\uB294 \uC720\uCD9C\uB41C \uAE30\uB85D\uC774 \uC788\uC5B4 \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uC5B4\uC694. \uB2E4\uB978 \uBE44\uBC00\uBC88\uD638\uB97C \uC0AC\uC6A9\uD574 \uC8FC\uC138\uC694.",
    form_password_pwned__sign_in: "\uC774 \uBE44\uBC00\uBC88\uD638\uB294 \uC720\uCD9C\uB41C \uBE44\uBC00\uBC88\uD638\uC608\uC694. \uBE44\uBC00\uBC88\uD638\uB97C \uC7AC\uC124\uC815\uD574 \uC8FC\uC138\uC694.",
    form_password_size_in_bytes_exceeded: "\uBE44\uBC00\uBC88\uD638\uAC00 \uD5C8\uC6A9\uB418\uB294 \uCD5C\uB300 \uBC14\uC774\uD2B8 \uC218\uB97C \uCD08\uACFC\uD588\uC5B4\uC694. \uBE44\uBC00\uBC88\uD638\uB97C \uC904\uC774\uAC70\uB098 \uC77C\uBD80 \uD2B9\uC218 \uBB38\uC790\uB97C \uC81C\uAC70\uD574 \uC8FC\uC138\uC694.",
    form_password_untrusted__sign_in: "\uBE44\uBC00\uBC88\uD638\uAC00 \uC720\uCD9C\uB410\uC744 \uC218 \uC788\uC5B4\uC694. \uACC4\uC815\uC744 \uBCF4\uD638\uD558\uAE30 \uC704\uD574 \uB2E4\uB978 \uBC29\uBC95\uC73C\uB85C \uB85C\uADF8\uC778\uD574 \uC8FC\uC138\uC694. \uB85C\uADF8\uC778 \uD6C4 \uBE44\uBC00\uBC88\uD638\uB97C \uC7AC\uC124\uC815\uD574\uC57C \uD574\uC694.",
    form_password_validation_failed: "\uC798\uBABB\uB41C \uBE44\uBC00\uBC88\uD638",
    form_username_invalid_character: void 0,
    form_username_invalid_length: "\uC544\uC774\uB514\uB294 {{min_length}}~{{max_length}}\uC790\uC5EC\uC57C \uD574\uC694.",
    form_username_needs_non_number_char: "\uC544\uC774\uB514\uC5D0\uB294 \uC22B\uC790\uAC00 \uC544\uB2CC \uBB38\uC790\uAC00 \uD558\uB098 \uC774\uC0C1 \uD3EC\uD568\uB3FC\uC57C \uD574\uC694.",
    identification_deletion_failed: "\uB9C8\uC9C0\uB9C9 \uC2DD\uBCC4 \uC815\uBCF4\uB294 \uC0AD\uC81C\uD560 \uC218 \uC5C6\uC5B4\uC694.",
    not_allowed_access: "\uC774\uBA54\uC77C \uC8FC\uC18C \uB610\uB294 \uC804\uD654\uBC88\uD638\uB294 \uAC00\uC785\uC5D0 \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uC5B4\uC694. \uC774\uBA54\uC77C \uC8FC\uC18C\uC5D0 '+', '=', '#' \uB610\uB294 '.'\uC774 \uD3EC\uD568\uB410\uAC70\uB098 \uC784\uC2DC \uC774\uBA54\uC77C \uC11C\uBE44\uC2A4 \uB3C4\uBA54\uC778\uC774\uAC70\uB098, \uBA85\uC2DC\uC801\uC73C\uB85C \uC81C\uC678\uB41C \uACBD\uC6B0\uC608\uC694. \uC774 \uC624\uB958\uAC00 \uACC4\uC18D\uB418\uBA74 \uC9C0\uC6D0\uC5D0 \uBB38\uC758\uD574 \uC8FC\uC138\uC694.",
    organization_domain_blocked: void 0,
    organization_domain_common: void 0,
    organization_domain_exists_for_enterprise_connection: void 0,
    organization_membership_quota_exceeded: void 0,
    organization_minimum_permissions_needed: void 0,
    organization_not_found_or_unauthorized: "\uC774 \uC870\uC9C1\uC758 \uBA64\uBC84\uAC00 \uC544\uB2C8\uC5D0\uC694. \uB2E4\uB978 \uC870\uC9C1\uC744 \uC120\uD0DD\uD558\uAC70\uB098 \uC0C8\uB85C \uB9CC\uB4E4\uC5B4 \uC8FC\uC138\uC694.",
    organization_not_found_or_unauthorized_with_create_organization_disabled: "\uC774 \uC870\uC9C1\uC758 \uBA64\uBC84\uAC00 \uC544\uB2C8\uC5D0\uC694. \uB2E4\uB978 \uC870\uC9C1\uC744 \uC120\uD0DD\uD574 \uC8FC\uC138\uC694.",
    passkey_already_exists: "\uC774 \uAE30\uAE30\uC5D0 \uC774\uBBF8 \uD328\uC2A4\uD0A4\uAC00 \uB4F1\uB85D\uB418\uC5B4 \uC788\uC5B4\uC694.",
    passkey_not_supported: "\uC774 \uAE30\uAE30\uC5D0\uC11C\uB294 \uD328\uC2A4\uD0A4\uB97C \uC9C0\uC6D0\uD558\uC9C0 \uC54A\uC544\uC694.",
    passkey_pa_not_supported: "\uB4F1\uB85D\uC744 \uC704\uD574 \uD50C\uB7AB\uD3FC \uC778\uC99D\uAE30\uAC00 \uD544\uC694\uD558\uC9C0\uB9CC, \uC774 \uAE30\uAE30\uB294 \uC9C0\uC6D0\uD558\uC9C0 \uC54A\uC544\uC694.",
    passkey_registration_cancelled: "\uD328\uC2A4\uD0A4 \uB4F1\uB85D\uC774 \uCDE8\uC18C\uB418\uC5C8\uAC70\uB098 \uC2DC\uAC04\uC774 \uCD08\uACFC\uB410\uC5B4\uC694.",
    passkey_retrieval_cancelled: "\uD328\uC2A4\uD0A4 \uC778\uC99D\uC774 \uCDE8\uC18C\uB418\uC5C8\uAC70\uB098 \uC2DC\uAC04\uC774 \uCD08\uACFC\uB410\uC5B4\uC694.",
    passwordComplexity: {
      maximumLength: "{{length}} \uBCF4\uB2E4 \uC9E7\uC740 \uBB38\uC790\uC5F4",
      minimumLength: "{{length}} \uB610\uB294 \uADF8 \uC774\uC0C1\uC758 \uBB38\uC790\uC5F4",
      requireLowercase: "\uC18C\uBB38\uC790",
      requireNumbers: "\uC22B\uC790",
      requireSpecialCharacter: "\uD2B9\uC218\uBB38\uC790",
      requireUppercase: "\uB300\uBB38\uC790",
      sentencePrefix: "\uBE44\uBC00\uBC88\uD638\uC5D0\uB294 \uB2E4\uC74C\uC774 \uAF2D \uD3EC\uD568\uB3FC\uC57C \uD574\uC694"
    },
    phone_number_exists: "\uC774 \uC804\uD654\uBC88\uD638\uB294 \uC774\uBBF8 \uC0AC\uC6A9 \uC911\uC774\uC5D0\uC694. \uB2E4\uB978 \uBC88\uD638\uB97C \uC2DC\uB3C4\uD574 \uC8FC\uC138\uC694.",
    session_exists: "\uC774\uBBF8 \uB85C\uADF8\uC778 \uC911\uC774\uC5D0\uC694.",
    web3_missing_identifier: "Web3 \uC9C0\uAC11 \uD655\uC7A5 \uD504\uB85C\uADF8\uB7A8\uC744 \uCC3E\uC744 \uC218 \uC5C6\uC5B4\uC694. \uACC4\uC18D\uD558\uB824\uBA74 \uC124\uCE58\uD574 \uC8FC\uC138\uC694.",
    web3_signature_request_rejected: "\uC11C\uBA85 \uC694\uCCAD\uC744 \uAC70\uBD80\uD588\uC5B4\uC694. \uACC4\uC18D\uD558\uB824\uBA74 \uB2E4\uC2DC \uC2DC\uB3C4\uD574 \uC8FC\uC138\uC694.",
    web3_solana_signature_generation_failed: "\uC11C\uBA85\uC744 \uC0DD\uC131\uD558\uB294 \uB3D9\uC548 \uC624\uB958\uAC00 \uBC1C\uC0DD\uD588\uC5B4\uC694. \uACC4\uC18D\uD558\uB824\uBA74 \uB2E4\uC2DC \uC2DC\uB3C4\uD574 \uC8FC\uC138\uC694.",
    zxcvbn: {
      couldBeStronger: "\uBE44\uBC00\uBC88\uD638\uB294 \uAD1C\uCC2E\uC9C0\uB9CC \uB354 \uAC15\uD558\uAC8C \uB9CC\uB4E4 \uC218 \uC788\uC5B4\uC694. \uBB38\uC790\uB97C \uB354 \uCD94\uAC00\uD574 \uBCF4\uC138\uC694.",
      goodPassword: "\uC88B\uC544\uC694! \uD6CC\uB96D\uD55C \uBE44\uBC00\uBC88\uD638\uC608\uC694.",
      notEnough: "\uBE44\uBC00\uBC88\uD638\uAC00 \uCDA9\uBD84\uD788 \uC548\uC804\uD558\uC9C0 \uC54A\uC544\uC694.",
      suggestions: {
        allUppercase: "\uBAA8\uB4E0 \uAE00\uC790\uAC00 \uC544\uB2CC \uC77C\uBD80 \uAE00\uC790\uB9CC \uB300\uBB38\uC790\uB85C \uBC14\uAFD4 \uBCF4\uC138\uC694.",
        anotherWord: "\uB35C \uD754\uD55C \uB2E8\uC5B4\uB97C \uB354 \uCD94\uAC00\uD574 \uBCF4\uC138\uC694.",
        associatedYears: "\uB098\uC640 \uAD00\uB828\uB41C \uC5F0\uB3C4\uB294 \uD53C\uD558\uB294 \uAC8C \uC88B\uC544\uC694.",
        capitalization: "\uCCAB \uAE00\uC790 \uC678\uC5D0\uB3C4 \uB300\uBB38\uC790\uB97C \uC11E\uC5B4 \uBCF4\uC138\uC694.",
        dates: "\uB098\uC640 \uAD00\uB828\uB41C \uB0A0\uC9DC\uB098 \uC5F0\uB3C4\uB294 \uD53C\uD558\uB294 \uAC8C \uC88B\uC544\uC694.",
        l33t: "'a'\uC5D0\uC11C '@'\uC640 \uAC19\uC774 \uC608\uCE21 \uAC00\uB2A5\uD55C \uBB38\uC790 \uB300\uCCB4\uB97C \uD53C\uD558\uC138\uC694.",
        longerKeyboardPattern: "\uB354 \uAE34 \uD0A4\uBCF4\uB4DC \uD328\uD134\uC744 \uC4F0\uACE0 \uC785\uB825 \uBC29\uD5A5\uC744 \uBC14\uAFD4 \uBCF4\uC138\uC694.",
        noNeed: "\uAE30\uD638, \uC22B\uC790, \uB300\uBB38\uC790 \uC5C6\uC774\uB3C4 \uAC15\uD55C \uBE44\uBC00\uBC88\uD638\uB97C \uB9CC\uB4E4 \uC218 \uC788\uC5B4\uC694.",
        pwned: "\uC774 \uBE44\uBC00\uBC88\uD638\uB97C \uB2E4\uB978 \uACF3\uC5D0\uC11C\uB3C4 \uC4F4\uB2E4\uBA74 \uBCC0\uACBD\uD574 \uC8FC\uC138\uC694.",
        recentYears: "\uCD5C\uADFC \uC5F0\uB3C4\uB294 \uD53C\uD558\uC138\uC694.",
        repeated: "\uBC18\uBCF5\uB418\uB294 \uB2E8\uC5B4\uB098 \uBB38\uC790\uB294 \uD53C\uD558\uC138\uC694.",
        reverseWords: "\uC775\uC219\uD55C \uB2E8\uC5B4\uB97C \uAC70\uAFB8\uB85C \uC4F0\uC9C0 \uB9C8\uC138\uC694.",
        sequences: "\uD754\uD55C \uBB38\uC790 \uC21C\uC11C\uB294 \uD53C\uD558\uC138\uC694.",
        useWords: "\uC5EC\uB7EC \uB2E8\uC5B4\uB97C \uC4F0\uB418 \uD754\uD55C \uBB38\uAD6C\uB294 \uD53C\uD558\uC138\uC694."
      },
      warnings: {
        common: "\uC790\uC8FC \uC4F0\uB294 \uBE44\uBC00\uBC88\uD638\uC608\uC694.",
        commonNames: "\uD754\uD55C \uC774\uB984\uC774\uB098 \uC131\uC740 \uCD94\uCE21\uD558\uAE30 \uC26C\uC6CC\uC694.",
        dates: "\uB0A0\uC9DC\uB294 \uC27D\uAC8C \uCD94\uCE21\uB3FC\uC694.",
        extendedRepeat: '"abcabcabc" \uAC19\uC740 \uBC18\uBCF5 \uD328\uD134\uC740 \uC27D\uAC8C \uCD94\uCE21\uB3FC\uC694.',
        keyPattern: "\uC9E7\uC740 \uD0A4\uBCF4\uB4DC \uD328\uD134\uC740 \uCD94\uCE21\uD558\uAE30 \uC26C\uC6CC\uC694.",
        namesByThemselves: "\uB2E8\uC77C \uC774\uB984\uC774\uB098 \uC131\uC740 \uCD94\uCE21\uD558\uAE30 \uC26C\uC6CC\uC694.",
        pwned: "\uC778\uD130\uB137 \uC720\uCD9C\uB85C \uBE44\uBC00\uBC88\uD638\uAC00 \uB178\uCD9C\uB410\uC5B4\uC694.",
        recentYears: "\uCD5C\uADFC \uC5F0\uB3C4\uB294 \uCD94\uCE21\uD558\uAE30 \uC26C\uC6CC\uC694.",
        sequences: '"abc" \uAC19\uC740 \uD754\uD55C \uBB38\uC790 \uC21C\uC11C\uB294 \uCD94\uCE21\uD558\uAE30 \uC26C\uC6CC\uC694.',
        similarToCommon: "\uC790\uC8FC \uC4F0\uB294 \uBE44\uBC00\uBC88\uD638\uC640 \uBE44\uC2B7\uD574\uC694.",
        simpleRepeat: '"aaa" \uAC19\uC740 \uBC18\uBCF5 \uBB38\uC790\uB294 \uCD94\uCE21\uD558\uAE30 \uC26C\uC6CC\uC694.',
        straightRow: "\uD0A4\uBCF4\uB4DC \uD55C \uC904 \uD328\uD134\uC740 \uCD94\uCE21\uD558\uAE30 \uC26C\uC6CC\uC694.",
        topHundred: "\uC790\uC8FC \uC4F0\uB294 \uBE44\uBC00\uBC88\uD638\uC608\uC694.",
        topTen: "\uC544\uC8FC \uB9CE\uC774 \uC4F0\uB294 \uBE44\uBC00\uBC88\uD638\uC608\uC694.",
        userInputs: "\uAC1C\uC778 \uC815\uBCF4\uB098 \uD398\uC774\uC9C0 \uAD00\uB828 \uB370\uC774\uD130\uB294 \uD53C\uD558\uB294 \uAC8C \uC88B\uC544\uC694.",
        wordByItself: "\uD55C \uB2E8\uC5B4\uB9CC \uC4F0\uBA74 \uCD94\uCE21\uD558\uAE30 \uC26C\uC6CC\uC694."
      }
    }
  },
  userButton: {
    action__addAccount: "\uACC4\uC815 \uCD94\uAC00",
    action__closeUserMenu: "\uC0AC\uC6A9\uC790 \uBA54\uB274 \uB2EB\uAE30",
    action__manageAccount: "\uACC4\uC815 \uAD00\uB9AC",
    action__openUserMenu: "\uC0AC\uC6A9\uC790 \uBA54\uB274 \uC5F4\uAE30",
    action__signOut: "\uB85C\uADF8\uC544\uC6C3",
    action__signOutAll: "\uBAA8\uB4E0 \uACC4\uC815\uC5D0\uC11C \uB85C\uADF8\uC544\uC6C3"
  },
  userProfile: {
    apiKeysPage: {
      title: "API \uD0A4"
    },
    backupCodePage: {
      actionLabel__copied: "\uBCF5\uC0AC \uC644\uB8CC!",
      actionLabel__copy: "\uC804\uCCB4 \uBCF5\uC0AC",
      actionLabel__download: ".txt \uB2E4\uC6B4\uB85C\uB4DC",
      actionLabel__print: "\uC778\uC1C4",
      infoText1: "\uC774 \uACC4\uC815\uC5D0 \uBC31\uC5C5 \uCF54\uB4DC\uAC00 \uD65C\uC131\uD654\uB3FC\uC694.",
      infoText2: "\uBC31\uC5C5 \uCF54\uB4DC\uB294 \uBE44\uBC00\uB85C \uBCF4\uAD00\uD574 \uC8FC\uC138\uC694. \uB178\uCD9C\uC774 \uC758\uC2EC\uB418\uBA74 \uB2E4\uC2DC \uC0DD\uC131\uD560 \uC218 \uC788\uC5B4\uC694.",
      subtitle__codelist: "\uC548\uC804\uD558\uAC8C \uC800\uC7A5\uD558\uACE0 \uBE44\uBC00\uB85C \uC720\uC9C0\uD558\uC138\uC694.",
      successMessage: "\uC774\uC81C \uBC31\uC5C5 \uCF54\uB4DC\uAC00 \uD65C\uC131\uD654\uB410\uC5B4\uC694. \uC778\uC99D \uC7A5\uCE58\uC5D0 \uC811\uADFC\uD560 \uC218 \uC5C6\uC744 \uB54C \uC774 \uC911 \uD558\uB098\uB85C \uB85C\uADF8\uC778\uD560 \uC218 \uC788\uC5B4\uC694. \uAC01 \uCF54\uB4DC\uB294 \uD55C \uBC88\uB9CC \uC0AC\uC6A9\uD560 \uC218 \uC788\uC5B4\uC694.",
      successSubtitle: "\uC778\uC99D \uC7A5\uCE58\uC5D0 \uC811\uADFC\uD560 \uC218 \uC5C6\uC744 \uB54C \uC774 \uC911 \uD558\uB098\uB85C \uB85C\uADF8\uC778\uD560 \uC218 \uC788\uC5B4\uC694.",
      title: "\uBC31\uC5C5 \uCF54\uB4DC \uC778\uC99D \uCD94\uAC00",
      title__codelist: "\uBC31\uC5C5 \uCF54\uB4DC"
    },
    billingPage: {
      paymentHistorySection: {
        empty: "\uACB0\uC81C \uB0B4\uC5ED\uC774 \uC5C6\uC5B4\uC694",
        notFound: "\uACB0\uC81C \uC2DC\uB3C4\uB97C \uCC3E\uC744 \uC218 \uC5C6\uC5B4\uC694",
        tableHeader__amount: "\uAE08\uC561",
        tableHeader__date: "\uB0A0\uC9DC",
        tableHeader__status: "\uC0C1\uD0DC"
      },
      paymentMethodsSection: {
        actionLabel__default: "\uAE30\uBCF8\uC73C\uB85C \uC124\uC815",
        actionLabel__remove: "\uC0AD\uC81C",
        add: "\uC0C8 \uACB0\uC81C \uC218\uB2E8 \uCD94\uAC00",
        addSubtitle: "\uACC4\uC815\uC5D0 \uC0C8 \uACB0\uC81C \uC218\uB2E8\uC744 \uCD94\uAC00\uD558\uC138\uC694.",
        cancelButton: "\uCDE8\uC18C",
        formButtonPrimary__add: "\uACB0\uC81C \uC218\uB2E8 \uCD94\uAC00",
        formButtonPrimary__pay: "{{amount}} \uACB0\uC81C",
        payWithTestCardButton: "\uD14C\uC2A4\uD2B8 \uCE74\uB4DC\uB85C \uACB0\uC81C",
        removeMethod: {
          messageLine1: "{{identifier}}\uAC00 \uC774 \uACC4\uC815\uC5D0\uC11C \uC0AD\uC81C\uB3FC\uC694.",
          messageLine2: "\uC774 \uACB0\uC81C \uC218\uB2E8\uC740 \uB354 \uC774\uC0C1 \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uACE0, \uC5F0\uACB0\uB41C \uAD6C\uB3C5\uB3C4 \uC791\uB3D9\uD558\uC9C0 \uC54A\uC544\uC694.",
          successMessage: "{{paymentMethod}}\uC774(\uAC00) \uACC4\uC815\uC5D0\uC11C \uC0AD\uC81C\uB410\uC5B4\uC694.",
          title: "\uACB0\uC81C \uC218\uB2E8 \uC0AD\uC81C"
        },
        title: "\uACB0\uC81C \uC218\uB2E8"
      },
      start: {
        headerTitle__payments: "\uACB0\uC81C",
        headerTitle__plans: "\uD50C\uB79C",
        headerTitle__statements: "\uBA85\uC138\uC11C",
        headerTitle__subscriptions: "\uAD6C\uB3C5"
      },
      statementsSection: {
        empty: "\uD45C\uC2DC\uD560 \uBA85\uC138\uC11C\uAC00 \uC5C6\uC5B4\uC694",
        itemCaption__paidForPlan: "{{plan}} {{period}} \uD50C\uB79C \uACB0\uC81C",
        itemCaption__proratedCredit: "\uC774\uC804 \uAD6C\uB3C5 \uC77C\uBD80 \uC0AC\uC6A9\uC5D0 \uB300\uD55C \uBE44\uB840 \uD06C\uB808\uB527",
        itemCaption__subscribedAndPaidForPlan: "{{plan}} {{period}} \uD50C\uB79C\uC744 \uAD6C\uB3C5\uD558\uACE0 \uACB0\uC81C\uD588\uC5B4\uC694",
        notFound: "\uBA85\uC138\uC11C\uB97C \uCC3E\uC744 \uC218 \uC5C6\uC5B4\uC694",
        tableHeader__amount: "\uAE08\uC561",
        tableHeader__date: "\uB0A0\uC9DC",
        title: "\uBA85\uC138\uC11C",
        totalPaid: "\uCD1D \uACB0\uC81C"
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: "\uAD00\uB9AC",
        actionLabel__newSubscription: "\uD50C\uB79C \uAD6C\uB3C5\uD558\uAE30",
        actionLabel__switchPlan: "\uD50C\uB79C \uBCC0\uACBD",
        tableHeader__edit: "\uD3B8\uC9D1",
        tableHeader__plan: "\uD50C\uB79C",
        tableHeader__startDate: "\uC2DC\uC791\uC77C",
        title: "\uAD6C\uB3C5"
      },
      subscriptionsSection: {
        actionLabel__default: "\uAD00\uB9AC"
      },
      switchPlansSection: {
        title: "\uD50C\uB79C \uBCC0\uACBD"
      },
      title: "\uACB0\uC81C"
    },
    connectedAccountPage: {
      formHint: "\uACC4\uC815\uC744 \uC5F0\uACB0\uD560 \uC81C\uACF5\uC790\uB97C \uC120\uD0DD\uD558\uC138\uC694",
      formHint__noAccounts: "\uC0AC\uC6A9 \uAC00\uB2A5\uD55C \uC678\uBD80 \uACC4\uC815 \uC81C\uACF5\uC790\uAC00 \uC5C6\uC5B4\uC694.",
      removeResource: {
        messageLine1: "{{identifier}}\uAC00 \uC774 \uACC4\uC815\uC5D0\uC11C \uC81C\uAC70\uB420 \uC608\uC815\uC774\uC5D0\uC694.",
        messageLine2: "\uC774 \uC5F0\uACB0\uB41C \uACC4\uC815\uC740 \uB354 \uC774\uC0C1 \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uACE0, \uAD00\uB828 \uAE30\uB2A5\uB3C4 \uC791\uB3D9\uD558\uC9C0 \uC54A\uC544\uC694.",
        successMessage: "{{connectedAccount}}\uAC00 \uACC4\uC815\uC5D0\uC11C \uC0AD\uC81C\uB410\uC5B4\uC694.",
        title: "\uC5F0\uACB0\uB41C \uACC4\uC815 \uC81C\uAC70"
      },
      socialButtonsBlockButton: "{{provider|titleize}} \uACC4\uC815 \uC5F0\uACB0",
      successMessage: "\uC774 \uC81C\uACF5\uC790\uAC00 \uACC4\uC815\uC5D0 \uCD94\uAC00\uB410\uC5B4\uC694.",
      title: "\uC5F0\uACB0\uB41C \uACC4\uC815 \uCD94\uAC00\uD558\uAE30"
    },
    deletePage: {
      actionDescription: "\uACC4\uC18D\uD558\uB824\uBA74 \uC544\uB798\uC5D0 \uACC4\uC815 \uC0AD\uC81C\uB97C \uC785\uB825\uD558\uC138\uC694.",
      confirm: "\uACC4\uC815 \uC0AD\uC81C",
      messageLine1: "\uACC4\uC815\uC744 \uC0AD\uC81C\uD560\uAE4C\uC694? \uC77C\uBD80 \uAD00\uB828 \uB370\uC774\uD130\uB294 \uBCF4\uAD00\uB420 \uC218 \uC788\uC5B4\uC694. \uC804\uCCB4 \uB370\uC774\uD130 \uC0AD\uC81C\uB97C \uC694\uCCAD\uD558\uB824\uBA74 \uACE0\uAC1D \uC9C0\uC6D0\uC5D0 \uBB38\uC758\uD574 \uC8FC\uC138\uC694.",
      messageLine2: "\uC774 \uC791\uC5C5\uC740 \uB418\uB3CC\uB9B4 \uC218 \uC5C6\uC5B4\uC694.",
      title: "\uACC4\uC815 \uC0AD\uC81C"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "\uC774 \uC774\uBA54\uC77C \uC8FC\uC18C\uB85C \uC778\uC99D \uCF54\uB4DC\uAC00 \uD3EC\uD568\uB41C \uC774\uBA54\uC77C\uC774 \uC804\uC1A1\uB3FC\uC694.",
        formSubtitle: "{{identifier}}\uC5D0\uAC8C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694",
        formTitle: "\uC778\uC99D \uCF54\uB4DC",
        resendButton: "\uCF54\uB4DC \uC7AC\uC804\uC1A1",
        successMessage: "{{identifier}} \uC774\uBA54\uC77C\uC774 \uACC4\uC815\uC5D0 \uCD94\uAC00\uB410\uC5B4\uC694."
      },
      emailLink: {
        formHint: "\uC778\uC99D \uB9C1\uD06C\uAC00 \uD3EC\uD568\uB41C \uC774\uBA54\uC77C\uC774 \uC774 \uC774\uBA54\uC77C \uC8FC\uC18C\uB85C \uC804\uC1A1\uB3FC\uC694.",
        formSubtitle: "{{identifier}}\uC5D0\uAC8C \uC804\uC1A1\uB41C \uC774\uBA54\uC77C\uC758 \uB9C1\uD06C\uB97C \uD074\uB9AD\uD574 \uC8FC\uC138\uC694.",
        formTitle: "\uC778\uC99D \uB9C1\uD06C",
        resendButton: "\uB9C1\uD06C \uC7AC\uC804\uC1A1",
        successMessage: "{{identifier}} \uC774\uBA54\uC77C\uC774 \uACC4\uC815\uC5D0 \uCD94\uAC00\uB410\uC5B4\uC694."
      },
      enterpriseSSOLink: {
        formButton: "\uB85C\uADF8\uC778 \uACC4\uC18D\uD558\uAE30",
        formSubtitle: "{{identifier}}\uB85C \uB85C\uADF8\uC778 \uC644\uB8CC"
      },
      formHint: "\uC774 \uC774\uBA54\uC77C \uC8FC\uC18C\uB97C \uCD94\uAC00\uD558\uB824\uBA74 \uBA3C\uC800 \uC778\uC99D\uD574\uC57C \uD574\uC694.",
      removeResource: {
        messageLine1: "{{identifier}} \uC774\uBA54\uC77C\uC774 \uC774 \uACC4\uC815\uC5D0\uC11C \uC81C\uAC70\uB420 \uC608\uC815\uC774\uC5D0\uC694.",
        messageLine2: "\uB354 \uC774\uC0C1 \uC774 \uC774\uBA54\uC77C \uC8FC\uC18C\uB85C \uB85C\uADF8\uC778\uD560 \uC218 \uC5C6\uC5B4\uC694.",
        successMessage: "{{emailAddress}} \uC774\uBA54\uC77C\uC774 \uACC4\uC815\uC5D0\uC11C \uC0AD\uC81C\uB410\uC5B4\uC694.",
        title: "\uC774\uBA54\uC77C \uC81C\uAC70"
      },
      title: "\uC774\uBA54\uC77C \uC8FC\uC18C \uCD94\uAC00",
      verifyTitle: "\uC774\uBA54\uC77C \uC8FC\uC18C \uC778\uC99D"
    },
    formButtonPrimary__add: "\uCD94\uAC00",
    formButtonPrimary__continue: "\uACC4\uC18D",
    formButtonPrimary__finish: "\uC644\uB8CC",
    formButtonPrimary__remove: "\uC0AD\uC81C",
    formButtonPrimary__save: "\uC800\uC7A5",
    formButtonReset: "\uCDE8\uC18C",
    mfaPage: {
      formHint: "\uCD94\uAC00\uD560 \uBC29\uBC95\uC744 \uC120\uD0DD\uD574 \uC8FC\uC138\uC694.",
      title: "2\uB2E8\uACC4 \uC778\uC99D \uCD94\uAC00"
    },
    mfaPhoneCodePage: {
      backButton: "\uAE30\uC874 \uBC88\uD638 \uC0AC\uC6A9",
      primaryButton__addPhoneNumber: "\uD734\uB300\uD3F0 \uBC88\uD638 \uCD94\uAC00",
      removeResource: {
        messageLine1: "{{identifier}}\uB294 \uB85C\uADF8\uC778\uD560 \uB54C \uB354 \uC774\uC0C1 \uC778\uC99D \uCF54\uB4DC\uB97C \uBC1B\uC9C0 \uC54A\uC544\uC694.",
        messageLine2: "\uACC4\uC815\uC774 \uC548\uC804\uD558\uC9C0 \uC54A\uC744 \uC218 \uC788\uC5B4\uC694. \uACC4\uC18D\uD560\uAE4C\uC694?",
        successMessage: "{{mfaPhoneCode}}\uC5D0 \uB300\uD55C SMS \uCF54\uB4DC 2\uB2E8\uACC4 \uC778\uC99D\uC774 \uC81C\uAC70\uB410\uC5B4\uC694.",
        title: "2\uB2E8\uACC4 \uC778\uC99D \uC81C\uAC70"
      },
      subtitle__availablePhoneNumbers: "SMS \uCF54\uB4DC 2\uB2E8\uACC4 \uC778\uC99D\uC744 \uC704\uD574 \uB4F1\uB85D\uD560 \uD734\uB300\uD3F0 \uBC88\uD638\uB97C \uC120\uD0DD\uD558\uC138\uC694.",
      subtitle__unavailablePhoneNumbers: "SMS \uCF54\uB4DC 2\uB2E8\uACC4 \uC778\uC99D\uC5D0 \uB4F1\uB85D\uD560 \uC218 \uC788\uB294 \uD734\uB300\uD3F0 \uBC88\uD638\uAC00 \uC5C6\uC5B4\uC694.",
      successMessage1: "\uB85C\uADF8\uC778\uD560 \uB54C \uC774 \uD734\uB300\uD3F0 \uBC88\uD638\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uCD94\uAC00\uB85C \uC785\uB825\uD574\uC57C \uD574\uC694.",
      successMessage2: "\uBC31\uC5C5 \uCF54\uB4DC\uB97C \uC548\uC804\uD55C \uACF3\uC5D0 \uBCF4\uAD00\uD574 \uC8FC\uC138\uC694. \uC778\uC99D \uAE30\uAE30\uC5D0 \uC811\uADFC\uD560 \uC218 \uC5C6\uAC8C \uB418\uBA74 \uBC31\uC5C5 \uCF54\uB4DC\uB85C \uB85C\uADF8\uC778\uD560 \uC218 \uC788\uC5B4\uC694.",
      successTitle: "SMS \uCF54\uB4DC \uC778\uC99D\uC774 \uD65C\uC131\uD654\uB410\uC5B4\uC694",
      title: "SMS \uCF54\uB4DC \uC778\uC99D \uCD94\uAC00"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "\uB300\uC2E0 QR \uCF54\uB4DC \uC2A4\uCE94 \uD558\uC138\uC694",
        buttonUnableToScan__nonPrimary: "QR \uCF54\uB4DC\uB97C \uC2A4\uCE94\uD560 \uC218 \uC5C6\uB098\uC694?",
        infoText__ableToScan: "\uC778\uC99D \uC571\uC5D0\uC11C \uC0C8 \uB85C\uADF8\uC778 \uBC29\uBC95\uC744 \uB9CC\uB4E4\uACE0 QR \uCF54\uB4DC\uB97C \uC2A4\uCE94\uD574 \uACC4\uC815\uC5D0 \uC5F0\uACB0\uD574 \uC8FC\uC138\uC694.",
        infoText__unableToScan: "\uC778\uC99D \uC7A5\uCE58\uC5D0\uC11C \uC0C8 \uB85C\uADF8\uC778 \uBC29\uBC95\uC744 \uB9CC\uB4E0 \uB2E4\uC74C \uC544\uB798 \uD0A4\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694.",
        inputLabel__unableToScan1: "\uC2DC\uAC04 \uAE30\uBC18 \uB610\uB294 \uC77C\uD68C\uC6A9 \uBE44\uBC00\uBC88\uD638\uAC00 \uCF1C\uC838 \uC788\uB294\uC9C0 \uD655\uC778\uD55C \uB4A4 \uACC4\uC815 \uC5F0\uACB0\uC744 \uC644\uB8CC\uD574 \uC8FC\uC138\uC694.",
        inputLabel__unableToScan2: "\uC778\uC99D \uC571\uC774 TOTP URI\uB97C \uC9C0\uC6D0\uD55C\uB2E4\uBA74 \uC804\uCCB4 URI\uB97C \uBCF5\uC0AC\uD574\uB3C4 \uB3FC\uC694."
      },
      removeResource: {
        messageLine1: "\uB85C\uADF8\uC778\uD560 \uB54C \uB354 \uC774\uC0C1 \uC774 \uC778\uC99D\uC790\uC758 \uC778\uC99D \uCF54\uB4DC\uAC00 \uD544\uC694\uD558\uC9C0 \uC54A\uC544\uC694.",
        messageLine2: "\uACC4\uC815\uC774 \uC548\uC804\uD558\uC9C0 \uC54A\uC744 \uC218 \uC788\uC5B4\uC694. \uACC4\uC18D\uD560\uAE4C\uC694?",
        successMessage: "\uC778\uC99D \uC571\uC744 \uD1B5\uD55C 2\uB2E8\uACC4 \uC778\uC99D\uC774 \uC81C\uAC70\uB410\uC5B4\uC694.",
        title: "2\uB2E8\uACC4 \uC778\uC99D \uC81C\uAC70"
      },
      successMessage: "\uC774\uC81C 2\uB2E8\uACC4 \uC778\uC99D\uC774 \uD65C\uC131\uD654\uB410\uC5B4\uC694. \uB85C\uADF8\uC778\uD560 \uB54C \uCD94\uAC00 \uB2E8\uACC4\uB85C \uC774 \uC778\uC99D\uC790\uC758 \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD574\uC57C \uD574\uC694.",
      title: "\uC778\uC99D \uC560\uD50C\uB9AC\uCF00\uC774\uC158 \uCD94\uAC00",
      verifySubtitle: "\uC778\uC99D \uC571\uC774 \uC0DD\uC131\uD55C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD574 \uC8FC\uC138\uC694",
      verifyTitle: "\uC778\uC99D \uCF54\uB4DC"
    },
    mobileButton__menu: "\uBA54\uB274",
    navbar: {
      account: "\uD504\uB85C\uD544",
      apiKeys: "API \uD0A4",
      billing: "\uACB0\uC81C",
      description: "\uACC4\uC815 \uC815\uBCF4\uB97C \uAD00\uB9AC\uD558\uC138\uC694.",
      security: "\uBCF4\uC548",
      title: "\uACC4\uC815"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: "{{name}}\uC774(\uAC00) \uC774 \uACC4\uC815\uC5D0\uC11C \uC0AD\uC81C\uB3FC\uC694.",
        title: "\uD328\uC2A4\uD0A4 \uC0AD\uC81C"
      },
      subtitle__rename: "\uD328\uC2A4\uD0A4 \uC774\uB984\uC744 \uBC14\uAFB8\uBA74 \uB354 \uC27D\uAC8C \uCC3E\uC744 \uC218 \uC788\uC5B4\uC694.",
      title__rename: "\uD328\uC2A4\uD0A4 \uC774\uB984 \uBCC0\uACBD"
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "\uC774\uC804 \uBE44\uBC00\uBC88\uD638\uB97C \uC0AC\uC6A9\uD588\uC744 \uC218 \uC788\uB294 \uB2E4\uB978 \uAE30\uAE30\uC5D0\uC11C \uB85C\uADF8\uC544\uC6C3\uD558\uB294 \uAC83\uC744 \uAD8C\uC7A5\uD574\uC694.",
      readonly: "\uAE30\uC5C5 \uC5F0\uACB0\uB85C\uB9CC \uB85C\uADF8\uC778\uD560 \uC218 \uC788\uC5B4 \uBE44\uBC00\uBC88\uD638\uB97C \uBCC0\uACBD\uD560 \uC218 \uC5C6\uC5B4\uC694.",
      successMessage__set: "\uBE44\uBC00\uBC88\uD638\uAC00 \uC124\uC815\uB410\uC5B4\uC694.",
      successMessage__signOutOfOtherSessions: "\uB2E4\uB978 \uBAA8\uB4E0 \uAE30\uAE30\uB294 \uB85C\uADF8\uC544\uC6C3\uB410\uC5B4\uC694.",
      successMessage__update: "\uBE44\uBC00\uBC88\uD638\uAC00 \uC5C5\uB370\uC774\uD2B8\uB410\uC5B4\uC694.",
      title__set: "\uBE44\uBC00\uBC88\uD638 \uC124\uC815",
      title__update: "\uBE44\uBC00\uBC88\uD638 \uBCC0\uACBD"
    },
    phoneNumberPage: {
      infoText: "\uC778\uC99D \uCF54\uB4DC\uAC00 \uD3EC\uD568\uB41C \uBB38\uC790 \uBA54\uC2DC\uC9C0\uAC00 \uC774 \uD734\uB300\uD3F0 \uBC88\uD638\uB85C \uC804\uC1A1\uB3FC\uC694. \uD1B5\uC2E0\uC0AC \uC694\uAE08\uC774 \uBD80\uACFC\uB420 \uC218 \uC788\uC5B4\uC694.",
      removeResource: {
        messageLine1: "{{identifier}}\uAC00 \uACC4\uC815\uC5D0\uC11C \uC81C\uAC70\uB420 \uC608\uC815\uC774\uC5D0\uC694.",
        messageLine2: "\uB354 \uC774\uC0C1 \uC774 \uD734\uB300\uD3F0 \uBC88\uD638\uB85C \uB85C\uADF8\uC778\uD560 \uC218 \uC5C6\uC5B4\uC694.",
        successMessage: "{{phoneNumber}}\uAC00 \uACC4\uC815\uC5D0\uC11C \uC81C\uAC70\uB410\uC5B4\uC694.",
        title: "\uD734\uB300\uD3F0 \uBC88\uD638 \uC81C\uAC70"
      },
      successMessage: "{{identifier}}\uAC00 \uACC4\uC815\uC5D0 \uCD94\uAC00\uB410\uC5B4\uC694.",
      title: "\uD734\uB300\uD3F0 \uBC88\uD638 \uCD94\uAC00",
      verifySubtitle: "{{identifier}}\uB85C \uC804\uC1A1\uB41C \uC778\uC99D \uCF54\uB4DC\uB97C \uC785\uB825\uD558\uC138\uC694",
      verifyTitle: "\uD734\uB300\uD3F0 \uBC88\uD638 \uC778\uC99D"
    },
    plansPage: {
      title: "\uD50C\uB79C"
    },
    profilePage: {
      fileDropAreaHint: "10MB\uBCF4\uB2E4 \uC791\uC740 JPG, PNG, GIF, WEBP \uC774\uBBF8\uC9C0\uB97C \uC5C5\uB85C\uB4DC\uD574 \uC8FC\uC138\uC694.",
      imageFormDestructiveActionSubtitle: "\uC774\uBBF8\uC9C0 \uC81C\uAC70",
      imageFormSubtitle: "\uC774\uBBF8\uC9C0 \uC5C5\uB85C\uB4DC",
      imageFormTitle: "\uD504\uB85C\uD544 \uC774\uBBF8\uC9C0",
      readonly: "\uAE30\uC5C5 \uC5F0\uACB0\uC5D0\uC11C \uC81C\uACF5\uB41C \uD504\uB85C\uD544 \uC815\uBCF4\uB77C \uC218\uC815\uD560 \uC218 \uC5C6\uC5B4\uC694.",
      successMessage: "\uD504\uB85C\uD544\uC774 \uC5C5\uB370\uC774\uD2B8\uB410\uC5B4\uC694.",
      title: "\uD504\uB85C\uD544 \uC5C5\uB370\uC774\uD2B8"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "\uC774 \uC7A5\uCE58\uC5D0\uC11C \uB85C\uADF8\uC544\uC6C3",
        title: "\uD65C\uC131\uD654\uB41C \uC7A5\uCE58"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "\uC7AC\uC2DC\uB3C4",
        actionLabel__reauthorize: "\uC9C0\uAE08 \uC778\uC99D\uD558\uAE30",
        destructiveActionTitle: "\uC81C\uAC70",
        primaryButton: "\uACC4\uC815 \uC5F0\uACB0\uD558\uAE30",
        subtitle__disconnected: "\uC774 \uACC4\uC815\uC740 \uC5F0\uACB0\uC774 \uD574\uC81C\uB410\uC5B4\uC694.",
        subtitle__reauthorize: "\uD544\uC694\uD55C \uAD8C\uD55C \uBC94\uC704\uAC00 \uC5C5\uB370\uC774\uD2B8\uB418\uC5B4 \uC77C\uBD80 \uAE30\uB2A5\uC774 \uC81C\uD55C\uB420 \uC218 \uC788\uC5B4\uC694. \uBB38\uC81C\uB97C \uD53C\uD558\uB824\uBA74 \uC774 \uC560\uD50C\uB9AC\uCF00\uC774\uC158\uC744 \uB2E4\uC2DC \uC778\uC99D\uD574 \uC8FC\uC138\uC694.",
        title: "\uC5F0\uACB0\uB41C \uACC4\uC815"
      },
      dangerSection: {
        deleteAccountButton: "\uACC4\uC815 \uC0AD\uC81C",
        title: "\uC704\uD5D8"
      },
      emailAddressesSection: {
        destructiveAction: "\uC774\uBA54\uC77C \uC8FC\uC18C \uC81C\uAC70",
        detailsAction__nonPrimary: "\uAE30\uBCF8\uC73C\uB85C \uC124\uC815",
        detailsAction__primary: "\uC778\uC99D \uC644\uB8CC",
        detailsAction__unverified: "\uC778\uC99D \uC644\uB8CC",
        primaryButton: "\uC774\uBA54\uC77C \uC8FC\uC18C \uCD94\uAC00",
        title: "\uC774\uBA54\uC77C \uC8FC\uC18C"
      },
      enterpriseAccountsSection: {
        title: "\uAE30\uC5C5 \uACC4\uC815"
      },
      headerTitle__account: "\uD504\uB85C\uD544",
      headerTitle__security: "\uBCF4\uC548",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "\uCF54\uB4DC \uC7AC\uC0DD\uC131",
          headerTitle: "\uBC31\uC5C5 \uCF54\uB4DC",
          subtitle__regenerate: "\uC0C8\uB85C\uC6B4 \uBCF4\uC548 \uBC31\uC5C5 \uCF54\uB4DC \uC138\uD2B8\uB97C \uBC1B\uC544 \uBCF4\uC138\uC694. \uC774\uC804 \uBC31\uC5C5 \uCF54\uB4DC\uB294 \uC0AD\uC81C\uB3FC \uC0AC\uC6A9\uD560 \uC218 \uC5C6\uC5B4\uC694.",
          title__regenerate: "\uBC31\uC5C5 \uCF54\uB4DC \uC7AC\uC0DD\uC131"
        },
        phoneCode: {
          actionLabel__setDefault: "\uAE30\uBCF8\uC73C\uB85C \uC124\uC815",
          destructiveActionLabel: "\uD734\uB300\uD3F0 \uBC88\uD638 \uC81C\uAC70"
        },
        primaryButton: "2\uB2E8\uACC4 \uC778\uC99D \uCD94\uAC00",
        title: "2\uB2E8\uACC4 \uC778\uC99D",
        totp: {
          destructiveActionTitle: "\uC81C\uAC70",
          headerTitle: "\uC778\uC99D \uC560\uD50C\uB9AC\uCF00\uC774\uC158"
        }
      },
      passkeysSection: {
        menuAction__destructive: "\uC0AD\uC81C",
        menuAction__rename: "\uC774\uB984 \uBCC0\uACBD",
        primaryButton: "\uD328\uC2A4\uD0A4 \uCD94\uAC00",
        title: "\uD328\uC2A4\uD0A4"
      },
      passwordSection: {
        primaryButton__setPassword: "\uBE44\uBC00\uBC88\uD638 \uC124\uC815",
        primaryButton__updatePassword: "\uBE44\uBC00\uBC88\uD638 \uBCC0\uACBD",
        title: "\uBE44\uBC00\uBC88\uD638"
      },
      phoneNumbersSection: {
        destructiveAction: "\uD734\uB300\uD3F0 \uBC88\uD638 \uC81C\uAC70",
        detailsAction__nonPrimary: "\uAE30\uBCF8\uC73C\uB85C \uC124\uC815",
        detailsAction__primary: "\uC778\uC99D \uC644\uB8CC",
        detailsAction__unverified: "\uC778\uC99D \uC644\uB8CC",
        primaryButton: "\uD734\uB300\uD3F0 \uBC88\uD638 \uCD94\uAC00",
        title: "\uD734\uB300\uD3F0 \uBC88\uD638"
      },
      profileSection: {
        primaryButton: "\uD504\uB85C\uD544 \uC218\uC815",
        title: "\uD504\uB85C\uD544"
      },
      usernameSection: {
        primaryButton__setUsername: "\uC544\uC774\uB514 \uC124\uC815",
        primaryButton__updateUsername: "\uC544\uC774\uB514 \uBCC0\uACBD",
        title: "\uC544\uC774\uB514"
      },
      web3WalletsSection: {
        destructiveAction: "\uC9C0\uAC11 \uC81C\uAC70",
        detailsAction__nonPrimary: "\uAE30\uBCF8\uC73C\uB85C \uC124\uC815",
        primaryButton: "\uC9C0\uAC11 \uC5F0\uACB0\uD558\uAE30",
        title: "Web3 \uC9C0\uAC11",
        web3SelectSolanaWalletScreen: {
          subtitle: "\uACC4\uC815\uC5D0 \uC5F0\uACB0\uD560 Solana \uC9C0\uAC11\uC744 \uC120\uD0DD\uD558\uC138\uC694.",
          title: "Solana \uC9C0\uAC11 \uCD94\uAC00"
        }
      }
    },
    usernamePage: {
      successMessage: "\uC544\uC774\uB514\uAC00 \uC5C5\uB370\uC774\uD2B8\uB410\uC5B4\uC694.",
      title__set: "\uC544\uC774\uB514 \uC5C5\uB370\uC774\uD2B8",
      title__update: "\uC544\uC774\uB514 \uC5C5\uB370\uC774\uD2B8"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}}\uAC00 \uC774 \uACC4\uC815\uC5D0\uC11C \uC81C\uAC70\uB420 \uC608\uC815\uC774\uC5D0\uC694.",
        messageLine2: "\uB354 \uC774\uC0C1 \uC774 Web3 \uC9C0\uAC11\uC73C\uB85C \uB85C\uADF8\uC778\uD560 \uC218 \uC5C6\uC5B4\uC694.",
        successMessage: "{{web3Wallet}}\uAC00 \uACC4\uC815\uC5D0\uC11C \uC81C\uAC70\uB410\uC5B4\uC694.",
        title: "web3 \uC9C0\uAC11 \uC81C\uAC70"
      },
      subtitle__availableWallets: "\uACC4\uC815\uC5D0 \uC5F0\uACB0\uD560 web3 \uC9C0\uAC11\uC744 \uC120\uD0DD\uD558\uC138\uC694.",
      subtitle__unavailableWallets: "\uC0AC\uC6A9 \uAC00\uB2A5\uD55C Web3 \uC9C0\uAC11\uC774 \uC5C6\uC5B4\uC694.",
      successMessage: "\uC9C0\uAC11\uC774 \uACC4\uC815\uC5D0 \uCD94\uAC00\uB410\uC5B4\uC694.",
      title: "web3 \uC9C0\uAC11 \uCD94\uAC00",
      web3WalletButtonsBlockButton: "{{provider|titleize}}"
    }
  },
  waitlist: {
    start: {
      actionLink: "\uB85C\uADF8\uC778",
      actionText: "\uC774\uBBF8 \uC561\uC138\uC2A4 \uAD8C\uD55C\uC774 \uC788\uC73C\uC2E0\uAC00\uC694?",
      formButton: "\uB300\uAE30 \uBAA9\uB85D\uC5D0 \uAC00\uC785",
      subtitle: "\uC774\uBA54\uC77C \uC8FC\uC18C\uB97C \uC785\uB825\uD558\uBA74 \uC900\uBE44\uB418\uB294 \uB300\uB85C \uC54C\uB824\uB4DC\uB9B4\uAC8C\uC694",
      title: "\uB300\uAE30 \uBAA9\uB85D\uC5D0 \uAC00\uC785"
    },
    success: {
      message: "\uACE7 \uC774\uB3D9\uB3FC\uC694...",
      subtitle: "\uC900\uBE44\uB418\uB294 \uB300\uB85C \uC5F0\uB77D\uB4DC\uB9B4\uAC8C\uC694",
      title: "\uB300\uAE30 \uBAA9\uB85D\uC5D0 \uD568\uAED8\uD574 \uC8FC\uC154\uC11C \uACE0\uB9C8\uC6CC\uC694!"
    }
  },
  web3SolanaWalletButtons: {
    connect: "{{walletName}}(\uC73C)\uB85C \uC5F0\uACB0",
    continue: "{{walletName}}(\uC73C)\uB85C \uACC4\uC18D",
    noneAvailable: 'Solana Web3 \uC9C0\uAC11\uC744 \uCC3E\uC744 \uC218 \uC5C6\uC5B4\uC694. Web3\uB97C \uC9C0\uC6D0\uD558\uB294 {{ solanaWalletsLink || link("wallet extension") }}\uC744(\uB97C) \uC124\uCE58\uD574 \uC8FC\uC138\uC694.'
  }
};
export {
  koKR
};
