// @clerk/clerk-expo/dist/hooks/useSignInWithApple.ios.js
"use strict";
var __create = Object.create;
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __getProtoOf = Object.getPrototypeOf;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toESM = (mod, isNodeMode, target) => (target = mod != null ? __create(__getProtoOf(mod)) : {}, __copyProps(
  // If the importer is in node compatibility mode or this is not an ESM
  // file that has been converted to a CommonJS file using a Babel-
  // compatible transform (i.e. "__esModule" has not been set), then set
  // "default" to the CommonJS "module.exports" for node compatibility.
  isNodeMode || !mod || !mod.__esModule ? __defProp(target, "default", { value: mod, enumerable: true }) : target,
  mod
));
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);
var useSignInWithApple_ios_exports = {};
__export(useSignInWithApple_ios_exports, {
  useSignInWithApple: () => useSignInWithApple
});
module.exports = __toCommonJS(useSignInWithApple_ios_exports);
var import_clerk_react = require("@clerk/clerk-react");
var import_errors = require("../utils/errors");
function useSignInWithApple() {
  const { signIn, setActive, isLoaded: isSignInLoaded } = (0, import_clerk_react.useSignIn)();
  const { signUp, isLoaded: isSignUpLoaded } = (0, import_clerk_react.useSignUp)();
  async function startAppleAuthenticationFlow(startAppleAuthenticationFlowParams) {
    if (!isSignInLoaded || !isSignUpLoaded) {
      return {
        createdSessionId: null,
        signIn,
        signUp,
        setActive
      };
    }
    let AppleAuthentication;
    let Crypto;
    try {
      [AppleAuthentication, Crypto] = await Promise.all([import("expo-apple-authentication"), import("expo-crypto")]);
    } catch {
      return import_errors.errorThrower.throw(
        "expo-apple-authentication is required to use Sign in with Apple. Please install it by running: npx expo install expo-apple-authentication expo-crypto"
      );
    }
    const isAvailable = await AppleAuthentication.isAvailableAsync();
    if (!isAvailable) {
      return import_errors.errorThrower.throw("Apple Authentication is not available on this device.");
    }
    try {
      const nonce = Crypto.randomUUID();
      const credential = await AppleAuthentication.signInAsync({
        requestedScopes: [
          AppleAuthentication.AppleAuthenticationScope.FULL_NAME,
          AppleAuthentication.AppleAuthenticationScope.EMAIL
        ],
        nonce
      });
      const { identityToken } = credential;
      if (!identityToken) {
        return import_errors.errorThrower.throw("No identity token received from Apple Sign-In.");
      }
      await signIn.create({
        strategy: "oauth_token_apple",
        token: identityToken
      });
      const userNeedsToBeCreated = signIn.firstFactorVerification.status === "transferable";
      if (userNeedsToBeCreated) {
        await signUp.create({
          transfer: true,
          unsafeMetadata: startAppleAuthenticationFlowParams == null ? void 0 : startAppleAuthenticationFlowParams.unsafeMetadata
        });
        return {
          createdSessionId: signUp.createdSessionId,
          setActive,
          signIn,
          signUp
        };
      }
      return {
        createdSessionId: signIn.createdSessionId,
        setActive,
        signIn,
        signUp
      };
    } catch (error) {
      if (error && typeof error === "object" && "code" in error && error.code === "ERR_REQUEST_CANCELED") {
        return {
          createdSessionId: null,
          setActive,
          signIn,
          signUp
        };
      }
      throw error;
    }
  }
  return {
    startAppleAuthenticationFlow
  };
}
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  useSignInWithApple
});
